%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% This is a (brief) model paper using the achemso class
%% The document class accepts keyval options, which should include
%% the target journal and optionally the manuscript type.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\documentclass[journal=jpcbfk,manuscript=article]{achemso}

%\documentclass[english,aps,preprint,pre,floatfix,nofootinbib,showpacs,showkeys]{revtex4-1}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Place any additional packages needed here.  Only include packages
%% which are essential, to avoid problems later. Do NOT use any
%% packages which require e-TeX (for example etoolbox): the e-TeX
%% extensions are not currently available on the ACS conversion
%% servers.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\usepackage[version=3]{mhchem} % Formula subscripts using \ce{}

\SectionNumbersOn

\usepackage{amsmath}
\usepackage{amssymb}
\usepackage{color}
\usepackage{CJK}
\usepackage{framed}
\usepackage[hidelinks]{hyperref}
\usepackage{algorithm}
\usepackage{algpseudocode}
\algdef{SE}[DOWHILE]{Do}{doWhile}{\algorithmicdo}[1]{\algorithmicwhile\ #1}

\usepackage[scaled=0.85]{beramono}
\usepackage[T1]{fontenc}
\usepackage{changepage}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% If issues arise when submitting your manuscript, you may want to
%% un-comment the next line.  This provides information on the
%% version of every file you have used.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%\listfiles

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Place any additional macros here.  Please use \newcommand* where
%% possible, and avoid layout-changing macros (which are not used
%% when typesetting).
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\makeatletter
    \setlength\@fptop{0\p@}
\makeatother

\newcommand*\mycommand[1]{\texttt{\emph{#1}}}

\newcommand{\state}[1]{$\mathcal{S}_{#1}$}

%\newcommand*{\rood}[1]{\textcolor{red}{#1}}
\newcommand*{\rood}[1]{#1}
%\newcommand*{\blauw}[1]{\textcolor{blue}{#1}}
\newcommand*{\blauw}[1]{#1}
%\newcommand*{\groen}[1]{\textcolor{green}{#1}}
\newcommand*{\groen}[1]{#1}
%\newcommand*{\blauwr}[1]{\textcolor{blue}{#1}}
\newcommand*{\blauwr}[1]{#1}

%\newcommand*{\addref}[1]{\textcolor{red}{\{ADD REF: #1\}}}

\newcommand*{\noter}[1]{\textcolor{red}{[[#1]]}}		% notes on
%\newcommand*{\noter}[1]{}					% notes off

%\usepackage[draft]{todonotes}   % notes shown
\usepackage[disable]{todonotes} % notes hidden
\usepackage{comment}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Meta-data block
%% ---------------
%% Each author should be given as a separate \author command.
%%
%% Corresponding authors should have an e-mail given after the author
%% name as an \email command. Phone and fax numbers can be given
%% using \phone and \fax, respectively; this information is optional.
%%
%% The affiliation of authors is given after the authors; each
%% \affiliation command applies to all preceding authors not already
%% assigned an affiliation.
%%
%% The affiliation takes an option argument for the short name.  This
%% will typically be something like "University of Somewhere".
%%
%% The \altaffiliation macro should be used for new address, etc.
%% On the other hand, \alsoaffiliation is used on a per author basis
%% when authors are associated with multiple institutions.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\author{Mike Jones}
\affiliation{%
  Pritzker School of Molecular Engineering, %
  The University of Chicago, %
  929 East 57th Street, Chicago, Illinois 60637, United States%
}

\author{Brennan Ashwood}
\affiliation{%
  Department of Chemistry, Institute for Biophysical Dynamics, and James Franck Institute, %
  The University of Chicago, %
  929 East 57th Street, Chicago, Illinois 60637, United States%
}

\author{Andrei Tokmakoff}
\affiliation{%
  Department of Chemistry, Institute for Biophysical Dynamics, and James Franck Institute, %
  The University of Chicago, %
  929 East 57th Street, Chicago, Illinois 60637, United States%
}

\author{Andrew L. Ferguson}
\email{andrewferguson@uchicago.edu}
\affiliation{%
  Pritzker School of Molecular Engineering, %
  The University of Chicago, %
  929 East 57th Street, Chicago, Illinois 60637, United States%
}

%\author{I. Ken Groupleader}
%\altaffiliation{A shared footnote}
%\email{i.k.groupleader@unknown.uu}
%\phone{+123 (0)123 4445556}
%\fax{+123 (0)123 4445557}
%\affiliation[Unknown University]
%{Department of Chemistry, Unknown University, Unknown Town}
%\alsoaffiliation[Second University]
%{Department of Chemistry, Second University, Nearby Town}

%\author{Susanne K. Laborator}
%\email{s.k.laborator@bigpharma.co}
%\affiliation[BigPharma]
%{Lead Discovery, BigPharma, Big Town, USA}

%\author{Kay T. Finally}
%\affiliation[Unknown University]
%{Department of Chemistry, Unknown University, Unknown Town}
%\alsoaffiliation[Second University]
%{Department of Chemistry, Second University, Nearby Town}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% The document title should be given as usual. Some journals require
%% a running title from the author: this should be supplied as an
%% optional argument to \title.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\title[]{TBA}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Some journals require a list of abbreviations or keywords to be
%% supplied. These should be set up here, and will be printed after
%% the title and author information, if needed.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\abbreviations{IR -- infrared, NMR -- nuclear magnetic resonance, UV -- ultraviolet}
%\keywords{Takens}

\begin{document}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% The manuscript does not need to include \maketitle, which is
%% executed automatically.  The document should begin with an
%% abstract, if appropriate.  If one is given and should not be, the
%% contents will be gobbled.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\newpage

\begin{abstract}

\noindent \url{https://www.overleaf.com/project/5e9e5110c524b8000192c548}

\end{abstract}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Start the main part of the manuscript here.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\newpage

\section{\label{sec:intro}Introduction}

Over the last couple decades, DNA has proved to be much more than a vessel for genetic information. From sensing, to computing, to directed self-assembly, the programmable and predictable nature of DNA has unlocked numerous unforeseen applications \citep{Seeman2017DNANanotechnology, Adleman1994MolecularProblems.pdf, Rothemund2006FoldingPatterns, Gu2010ALine}. Recently, structural DNA nanotechnology has enabled self-assembly on micro to milli scales, and dynamic DNA nanotechnology has been used to perform basic calculation and to probe single molecules via temporal DNA signatures\citep{MhatreV.HoJi-AnnLee2012NIHAccess, Bui2018, Shah2019}. \noter{Please correct reference ``MhatreV.HoJi-AnnLee2012NIHAccess''} Both technologies rely on the hybridization reaction between complementary DNA strands and leverage the flexibility of shorter DNA oligomers to participate in these reactions. Although many experimental and computational studies have rigorously explored DNA dynamical phenomena such as hybridization, hairpin formation, and single base pair flipping, the sequence-dependent mechanisms of hybridization and dissociation dynamics are not fully understood \citep{Yin2011KineticsHybridization, Xiao2019, Hinckley2014Coarse-grainedEffects, Sanstead2016, Porschke1971CooperativeTransition, Porschke1973ThermodynamicsPairs, Chen2007InfluenceHybridization, Craig1971ElaxationOligon}. Moreover, it is unclear the extent to which these processes evolve in an "all-or-nothing" fashion or if an ensemble meta-stable states facilitates the transition. Recent breakthrough studies have coupled experimental techniques with machine learning and MD simulations to investigate and predict sequence-dependent kinetics \citep{Schickinger2018TetheredHelices, Zhang2018PredictingSequence}. Where these studies focus on association and dissociation kinetics alone, we broaden our analysis into higher order dynamical processes and meta-stable intermediates. The stability of certain intermediates states, such as out-of-register or  base pairing in repetitive sequences, has been well documented in previous computational studies \citep{Phys2014, Xiao2019}). Furthermore, frayed structures and dynamics have been investigated in numerous computational and experimental studies \citep{Zgarbova2014BaseRNA, Nonin1995TerminalFraying, Nikolova2012ProbingSimulations, Andreatta2006UltrafastHelix}. Sandstead et al. highlighted the role of these dynamics during duplex dissociation, where the stability of frayed states was dictated by G:C bond placement in the 10-mer oligonucleotide sequence \citep{Sanstead2016}. In this work, we study the same four sequences explored by Sandstead et al in an effort to uncover the sequence-dependent dynamics and their relation to  metastable structures mentioned above.

\textcolor{purple}{I would suggest re-organizing this paragraph to emphasize how the dynamic insight from various experiments is limited and therefore high-level simulations are required to understand DNA dynamics in detail. Then, you can summarize why simulating DNA hybridization is difficult and the previously employed techniques} Given the long timescales on which DNA hybridization and dissociation events occur, the study of these processes are generally not amenable to direct simulation techniques \citep{Phys2014}. Instead, many previous studies of DNA hybridization have employed accelerated sampling methods such as umbrella sampling \citep{Schmitt2013ExploringSurface} transition path sampling \citep{Sambriski2009}, and forward flux sampling \citep{Schmitt2013ExploringSurface, Sambriski2009,  Hoefert2011MolecularOligonucleotides, Romano2013DNADependence}. Other computational studies use dramatically elevated temperature or denaturing solvent concentrations induce one-way dissociation events \citep{Wong2008TheSimulations, Perez2010Real-timeUnfolding}. Experimental studies often employ temperature-jump, salt-jump, pH-jump, or other perturbative methods to drive DNA out of equilibrium and monitor relaxation processes in one direction \citep{Morrison1993SensitiveSolution, Wetmur1968KineticsDNA, Craig1971ElaxationOligon, Porschke1973ThermodynamicsPairs, Williams1989LaserDGCATGC, Narayanan2012ExploringMixing, Chen2007InfluenceHybridization, Sanstead2018DirectDehybridization}. Single molecule diffusion and tethered multifluorophore assays facilitate equilibrium analysis, however these present technical difficulties and are hampered by data collection rates \citep{Liu20173DSolution,  Schickinger2018TetheredHelices, Chen2008Base-by-baseSpectroscopy, Dupuis2013Single-moleculeHelices}. Förster resonance energy transfer (FRET) analysis, particularly when coupled with methods mentioned above, provides additional resolution, but it unclear how fluorescent tags may interfere with the dynamics of short nucleotides \citep{Morrison1993SensitiveSolution}.

In this work, we use the coarse-grain 3 site per nucleotide (3spn2) model to simulate hybridization and dissociation behavior near each sequence's melting temperature \citep{Hinckley2013AnHybridization}.  We leverage the properties of Markov State Models (MSMs) -- namely that conditional probability depends only on the current state of the system \citep{Pande2010EverythingAsk} -- to combine many independent and unbiased trajectories and develop an understanding of sequence-specific kinetics and thermodynamics. MSMs have recently been implemented to study mechanisms and microstate distributions of DNA hybridization  \citep{Jin2019, Xiao2019}, but the slowest sequence-dependent kinetics were not the focus of these studies. \citet{Pinamonti2017} used MSMs to compare the slowest dynamics of short RNA nucleotides and found that stacking timescales are highly sequence dependent \citep{Pinamonti2017}. We take a similar approach to study 10-mer DNA oligonucleotides and introduce State Reversible Vampnets (SRVs) to directly learn the slowest sequence-dependent dynamical modes \citep{Chen}. Furthermore, we integrate SRVs into the MSM pipeline by generating an optimized low dimensional basis in which microstates clustering can be performed. We show that SRV coordinates can be useful for both directly interpreting dynamical trends and for improving overall SRV-MSM quality when compared to more conventional methods such as time-structure independent components analysis (tICA).

We find that GC base pair placement in decamer oligonucleotides has a substantial effect on dynamical behavior. By evaluating equilibrium trajectories we can study the relevance of meta-stable states during the hybridization and dissociation process. Furthermore we can perform these analyses without biasing simulations or assuming that one processes is a strictly reversible version of the another. Because SRVs generate an optimized low dimensional basis, we show that we can access higher resolution MSMs (shorter lag time) and generate more detailed models. Additionally, we can compare slow dynamical modes and meta-stable states between sequence-specific SRV-MSMs. Within these meta-stable dynamical states, we leverage diffusion maps to analyze the diversity of structures whose inter-conversion rate are too fast to produce unique slow modes. Taken together, our analysis reflects similar results to previous computational and experimental DNA work, while elucidating new insights into sequences dependent dynamics, meta-stable structures, and relative timescales. 

\section{\label{sec:methods}Methods}

\subsection{\label{sec:methods}3spn2 Model}

Is some additional information warranted on the model itself before going into this specific setup?

\subsection{\label{sec:methods}Simulation set up}

We initialized four sequences previously investigated by Sandstead et al. -- 5'ATATATATAT3' (AT-all), 5'GATATATATC3' (GC-end), 5'ATATGCATAT3' (GC-core), and 5'ATGATATCAT3' (GC-mix) -- along with their complementary strands according to 3SPN.2 documentation \citep{Sanstead2016, Phys2014}. We initialized explicit ions such that 240 mM NaCl and 18 mM MgCl2 \textcolor{purple}{So these are initially in the bulk and then may interact with the DNA? Are these the effective concentrations after you shrink the box?} were added to the box in addition to 18 Na counter ions to balance the charge from the 9 phosphate groups in each oligonucleotide backbone \citep{Hinckley2015}. In order to maximize concentration without allowing strands to see each other through periodic boundaries, we set the box size just larger than the sum of the maximum end-to-end extension length of a single strand and the force cutoff (using Ewald summation method is set at 20 A). This translated to a box size of 77.74 Å and an effective oligo concentration of 7 mM. We used an Ewald potential to calculate long range Coulombic interaction between DNA and ions. We maintained a Debye-Huckel screening potential with an ionic strength of 240 mM -- corresponding to NaCl concentration -- to account for phosphate backbone interactions. These electrostatics preserve the persistence length and intrinsic curvature of DNA while taking the effects of ion-DNA interactions\citep{Hinckley2015}. We run our simulations in the NVE ensemble and fix temperature via a Langevin thermostat in order to model implicit solvent interactions \citep{Schneider1978Molecular-dynamicsTransitions}. Simulation temperature was determined by sequence specific melting temperature such that strands were equally likely to spend time in the hybridized or dissociated state. Tmelt simulations were run for $3*10^{8}$ time steps, equating to 4.5$\mu$s simulation time, and frames were saved every 30 ps. A 15 fs timestep was used for all simulations runs. For each sequences, 100 simulations were performed in parallel, consuming about 32 serial cpu-hours of computation time for each independent simulation. An approximate Tmelt Boltzman distribution was replicated by initializing half of runs from the hybridized state and half from a random dissociated state. In order to allow for further equilibration, the first third (1.5 $\mu$s) of each simulation was removed  \textcolor{purple}{Why 1.5 microseconds? Is this arbitrary or is there a good reason to remove 1.5?}, resulting in 100 x 100000 frames and a total of 300$\mu$s simulation time per sequence.

\subsection{\label{sec:methods}Featurization}

All intermolecular pairwise distances for both oligonucleotides were calculated at each frame using the MDtraj software package \citep{McGibbon2015MDTraj:Trajectories}. Based on the self-complementary nature of each sequence, we averaged permutable distances (45 pairs in total) together. This permutation reduction follows a similar procedure used in TICAgg coordinates construction \citep{Sengupta2019AutomatedSelf-assembly}. The VAMP-2 scoring method \textcolor{purple}{Since the VAMP-2 scoring comes up a lot, can you maybe another sentence or two on how exactly this works? Citing the Noe paper helps, but I think experimentalists may get a bit lost with it}, which represents the sum of squared estimates for the transfer operator, was employed to evaluate the quality of this feature set \citep{Mardt2018VAMPnetsKinetics}. We compared the VAMP-2 score for the permutation-free distances to the complete set of intermolecular distances and found only a small loss in kinetic variance when using the reduced data set. Furthermore, we found an increase in VAMP-2 score when using reciprocal pairwise distances and chose these reciprocal permutation-free coordinates as a consistent feature set. These features were normalized and passed into sequence-specific SRVs. 

\begin{figure}[ht!]
	\begin{center}
        \includegraphics[width=\textwidth]{Figs/figs_0804/srv_crossval.png}
        \caption{5-fold cross validation procedure to select number of SRV coordinates. We look for both the converge of the VAMP-2 and consistent scores between folds to ensure that the model is not fitting on statistical noise. (probably will be SI, but could include AT-all cross-val in the walkthrough section)}
        \label{fig:srv_crossval}
	\end{center}
\end{figure}

\subsubsection{\label{sec:methods}SRVs}
 
SRVs were first developed by Chen et al. as a means to directly learn slow eigenfunctions of the transfer operator \citep{Chen}. The framework uses a twin-lobed artificial neural network, similar in structure to the network employed in VAMPnets \citep{Mardt2018VAMPnetsKinetics}, to learn a low dimensional representation of input features. This representation is optimized for the variational approach to conformational dynamics (VAC) from which the leading eigenfunctions of the transfer operator can be estimated (equation here?). The resulting orthogonal modes are associated with the slowest dynamical processes in a system, and can be used to interpret kinetic information directly (such as physical correlations and timescales) and to construct MSMs. Other methods, such as time-lagged independent component analysis (TICA) and kernel TICA (kTICA), have been employed in place of SRVs but suffer limitations from the linear transformation in the case of the former and high computational cost in the case of the latter \citep{Harrigan2017LandmarkDynamics}. SRVs provide robust nonlinear approximation and computation timescales linearly with the amount of input data. This is a key attribute to our system as 10 million frames with 55 features in each frame are used as input data. The SRV framework has been tested on toy systems where the true eigenfunctions of the transfer operator are known and on small protein simulation data such as the WW-domain and Trp-cage mini-protein \citep{Chen}, \citep{Sidky}. For these latter system, SRV-MSMs were contructed in order to find the stability of metastables states as well as transition probabilities between those states.

Using optimized hyperparameters and featurized trajectory data, we generated a hierarchical dynamic encoder (HDE) to transform 55 reciprocal pairwise distances into a low dimensional SRV feature set. In order to maintain consistency between sequences, we kept all SRV training hyperparameters the same with the exception of the number of outputed slow modes. We determined the number of slow mode via cross-validation on the VAMP-2 score to ensure that the coordinate did not over fit on statistical noise \citep{McGibbon2015VariationalKinetics}. In particular we looked for a diminishinghttps://www.overleaf.com/project/5eff37185f425e0001a920ca increase in the vamp-2 score, inconsistency between cross validation scores -- suggesting that the model maybe learning different modes, and a widening gap between train and test scores -- suggesting that the model might be overfitting on statistical noise in the training data. We used a batch size of 50000 and ran each model for total of 30 training epochs. We used two hidden layers and set the size of each layer to 100. For cross-validation and comparison between different hyper-parameters, we used a 80/20 validation split training. SRV training required about 22 GPU-minutes across 1 GPU and 10 CPUs. SRV training was implemented using Keras and Tensorflow \citep{KerasGithub.Com} \citep{Abadi2016TensorFlow:Systems}.

\subsubsection{\label{sec:methods}SRV-MSMs}

MSMs are a powerful tool for interpreting large amounts of simulation data in a statistically robust and experimentally comparable way. The technique relies on the discretitization of kinetically similar conformations into microstates and finds the conditional probability between states within some lag time. The reliance on conditional probabilities allows for many independent simulations (longer than the lag time) to be collectively interpreted. To take full advantage of the MSM frameworks, however, the input basis should be as kinetically meaningful as possible \citep{Pande2010EverythingAsk}. This becomes crucial in our system given the large difference in timescales between leading modes. Because SRV eigenfunctions translate simulation features into their slowest kinetic representations, they are optimally suited as an MSM basis. To prove this, SRV-MSM VAMP-2 scores were shown to be consistently higher than MSMs constructed from TICA coorinates (TICA-MSMs) \citep{Sidky}. Furthermore, SRV-MSM implied timescales converge faster than TICA-MSM timescales, enabling a shorter lag time and therefore a higher resolution model. To build our SRV-MSM framework, we employed the Pyemma MSM pipeline and generated independent models for each sequence \citep{Scherer2015PyEMMAModels}. After passing in SRV coordinates, we perform k-means microstate clustering, Bayesian MSM construction, and PCCA+ macrostate assignments. The number of microstates were determined by VAMP-2 score, and the SRV-MSM lag time was selected based on implied timescales convergence.  The number of PCCA+ macrostates was determined based on the characteristic of each system and will be discussed more in depth in the results. 

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\section{\label{sec:Results}Results}
- lead with same kind of diagram for each sequence, add additional figures when necessary for a given sequence

\subsection{\label{sec:Results}AT-all}

\subsubsection{\label{sec:Results}SRV optimization and analysis}

In our analysis, we found that the AT-all sequence, given its repetitive structure and lack of GC-content, produced the cleanest dynamics and displayed a canonical spectral gap between modes. For this reason, we lead our discussion with this sequence and use it as a case study to work through our SRV-MSM pipeline step-by-step. Our first task was to identify the SRV lag time that was longer than the intrinsic Markov timescales of the system, yet short enough to resolve the dynamics of interest \citep{Phys2018MarkovValidation}. We found that most implied timescales (with the exception of the leading timescale) converge at a lag time of 600 ns. We kept a looser constraint on the convergence of the leading mode as it reflects the less Markovian dynamic of strand association and dissociation \textcolor{purple}{Not sure how to describe this or if it warrants further discussion?} . It should be noted that our lag time selection was informed by the other sequences as well to maintain consistency. Next we selected an optimal number of SRV components to include in our analysis. After a certain point, higher order dynamical modes provide diminishing contributions the overall kinetic variance as measured by the Vamp-2 score, and the model can begin fitting on statistical noise in the trajectory data instead of the true dynamics \citep{McGibbon2015VariationalKinetics}. It is also more difficult to perform kmeans clustering on a high dimensional space, especially when those higher dimensions are less kinetically relevant \citep{Pande2010EverythingAsk}. For these reasons, the number of slow SRV components should be carefully selected based on the specific system of interest. As shown in (Figure \ref{fig:srv_crossval}), we see a clear convergence of the VAMP-2 score after five slow modes and select these modes as our optimized SRV basis. Next, we seek to interpret the physical relevance of these leading modes by plotting the Pearson correlation of each mode with the 100 intermolecular distances between strands. The quantitative meaning of these coordinates can be difficult to interpret given their nonlinear relationship to the SRV collective variables, but the relative different between these correlations shows which coordinates are most affected by (or effective on) each process. For example, the first slow mode shows a significant positive correlation to each distance and the strongest correlation with matching base pair distances (shown along the main diagonal). Given these relationships and the substantially longer timescale of this process, we can deduce that this leading mode corresponds to the dynamics of the overall hybridization and dissociation process. The next four SRV components all show a relatively high correlation along different offset diagonals. These diagonals correspond to the intermolecular distances between shifted base pairs and point to the existence of gradually faster shifting or "slithering" processes \textcolor{purple}{I suppose the GCGCGC sequence in the Wales paper provides an interesting comparison to AT-all
}. Slow slithering mechanisms have similarly been reported in simulation studies to occur on orders of magnitude longer timescales than underlying fast dynamics \citep{Markegard2015, Xiao2019, Maciejczyk2014DNAModel}. 

\subsubsection{\label{sec:Results}SRV-MSM construction and optimization}

From this analysis we can determine that the slowest dynamics are fully characterized by the hybridization process and slithering behavior of out of register base pairs. Although this is qualitatively informative, we can access a more holistic picture of sequence kinetics and thermodynamics by using these SRV coordinates as a basis on which to construct an MSM. Because these coordinates are already capturing a majority of the system's kinetic variance, they serve as an ideal basis on which to group frames into microstates. We performed k-means clustering, and optimized the number of microstates at 200 by monitoring VAMP-2 score. Next, we selected an MSM lag time in a similar fashion to our SRV lag time selection process. In Figure \ref{fig:AT-all_dynamic} we compare the convergence of SRV-MSM and TICA-MSM timescales, where SRV-MSM timescales converge consistently faster and to higher values of all leading modes. This enables us to select a shorter lag time and build a higher resolution model than we could from an analogous TICA basis. Using an MSM lag time of 1.2 ns, we then built a Bayesian MSM to calculate transition probability matrix between each microstate. Finally, PCCA+ spectral clustering was implemented to group these microstates into macrostates that each represent a collection of metastable structures. Previous work have used a common set of microstates and/or performed manual clustering of microstates based on physical read outs from simulation data (stacking score, energies, etc) \citep{Pinamonti2017PredictingModels}, \citep{PinamontiAnalyzedModels}. Although these techniques are useful for performing comparisons between sequences, we saw better results when optimizing MSMs to capture the most detail of sequence individually and thus developed an independent set of microstates and macrostates for each sequence. For AT-all, we kept to the convention of clustering into n+1 macrostates, where n is the number of slow components captured by the MSMs. To visualize these six macrostates, we project the data into the two leading TICA coordinates. Although SRVs outperform these coordinates for the purpose of MSM construction, TICA represent good high variance collective variables on which to visualize free energies and state assignments \citep{Sidky} \textcolor{purple}{Still not sure why you use tICA here, does not nonlinear nature make them harder to include physically? Should these be in SI}. It is clear in Figure \ref{fig:AT-all_in_one} that the PCCA state assignments are capturing free energy minima in TICA space as independent states. After assigning these macrostates we can then calculate their relative probabilities and free energies, visualize representative molecular renderings, and estimates transition probabilities between states. In this final step, we use a minimum flux cutoff of 2e-6 in order to mitigate erroneous quick transition or skipping between states. 

\begin{figure}[ht!]
	\begin{center}
        \includegraphics[width=\textwidth]{Figs/figs_0804/AT-all_dynamic.png}
        \caption{AT-all SRV-MSM timescales convergence and implied timescales using a lag time of 1.2 ns. Leading slow mode correlations to all 100 intermolecular distances between oligos.}
        \label{fig:AT-all_dynamic}
	\end{center}
\end{figure}

\subsubsection{\label{sec:Results}Implied timescales for TICA-MSMs for }

In our coarse-grained SRV-MSM, we observe an "aligned hybridized" state, dissociated state, and four "shifted" states characterized by different combinations of out-of-register base pairings. These shifted states consist of 2 or 4 base pair shifting (single or doubled shifted) in either the 5' or 3' direction with varied stability and transition fluxes between kinetically neighboring states. This state decomposition is expected given the repetitive nature of the AT-all sequence and previous computational results that find shifted conformations form "deep kinetic traps" \citep{Xiao2019}, \citep{Phys2014}. We found a substantial difference in thermodynamic stability and kinetic behavior between the 5'A shifted states and 3'T shifted states. This difference can be accounted for by examining experimental studies on the thermodynamics of "dangling ends" -- unpaired bases adjacent to the paired duplex -- and "inert tails" -- free bases that extend beyond the dangling end \citep{Michele2014EHybridization}. Dangling 5' ends have a consistently stabilizing effect, and inert tails decrease stability as they increase in length \textcolor{purple}{The effect of inert tails seems sequence dependent and not always destabilizing (Benight), especially at high ionic strength}. It has been shown that two base pair 5' dangling ends (or dangling ends with one inert tail) have higher melting temperature and are enthalpically favorable compared to 3' ends \citep{Senior1988InfluenceDuplexes, Dickman2012ThermodynamicDNAs} \textcolor{purple}{The Benight paper also shows that 3'-dangling ends and tails alter the change in heat capacity (dCp) upon hybridization much more than 5'. Additionally, their analysis suggests 3'-dangling ends lead to a reduction in bound counterions relative to 5'. Taken together, it would seem that 3'-dangling ends and tails may perturb the duplex structure/solvation environment in some manner. }. This is likely due to 5' ends preferentially stacking on the core duplex compared to 3' ends. Furthermore, DNA nearest neighbor (NN) calculations that include dangling end contributions (Figure \ref{fig:AT-all_in_one}) predict that the specific nearest neighbors bonds in the 5' shifted state are more energetically favorable than those in the 3' shifted state \textcolor{purple}{Also, it may be worth stating that NN parameters agree with experimental melting thermodynamics for the studied sequences (cite Paul's JACS paper). The NN parameters are occasionally quite bad for some sequences.} \citep{Santalucia2004TM, SantaLucia1998AThermodynamics} (Note that these calculations are performed at higher salt concentration, and that salt seems to have a significant effect on the impact of inert tails on stability) \textcolor{purple}{You may want to be more specific about how counterions impact inert tail thermodynamics  (more destabilizing at low counterion) and that your simulations are at fairly high counterion concentration}. Taken together, these experimental results substantiate why we observe the single-shifted 5' state to be only slightly less stable relative to the aligned hybridized state. Furthermore, we expect implied timescales of the two leading modes to be on the same order of magnitude as these reflect the average lifetimes of these states. The stability of the single-shifted 3' state is notably lower than that of the single-shifted 5' state. In fact, the free energy is about equal to that of the double-shifted 5' state, despite it having less intact base pairs. This is partially explained by nearest neighbor calculations, however these do not account for the differential effects of the 5' and 3' inert tails or kinetic considerations which we will discuss more below \textcolor{purple}{another inaccuracy of DNA NN is that they assume an all-or-nothing hybridization, so if the melting of each of these isolated conformations contained varying deviation from all-or-nothing behavior, then there would be another factor unaccounted for. 
}. The double-shifted 3' state is more stable than one might expect given the difference in stability between the single and double 5' shifted states. This might be attributed to the asymptotic effects of inert tail destabilitization -- longer tails tend to have a diminishing effect on overall strand stability \citep{Michele2014EHybridization}. Accordingly, the longer inert tails in the double-shifted states have less of an effect on relative 5' vs. 3' stability compared to the shorter tails in the single-shifted states.

\begin{figure}[ht!]
	\begin{center}
        \includegraphics[width=\textwidth]{Figs/figs_0804/AT-all_in_one.PNG}
        \caption{a) Thermo value calculated from Santalucia data and from MSMs b) TICA projections of freen energies and state clustering c) and transitions probabilities between states \textcolor{purple}{Consider getting rid of enthalpy and entropy columns (can add to SI). Need to add State numbers next to names}}
        \label{fig:AT-all_in_one}
	\end{center}
\end{figure}

\subsubsection{\label{sec:Results}Transition probabilities between states}

Beyond state probabilities and free energy approximations, the macrostate MSM also yields discrete kinetic information in the form of transition probabilities between states. Figure \ref{fig:AT-all_in_one} shows the probability of moving from one state to another within the MSM lag time (1.2 ns). All transition probabilities are higher when moving towards a more aligned state than towards a more shifted state, suggesting that these metastable shifted states play a more significant role in facilitating the hybridization process than dissociation. Furthermore, we see equal or higher transition rates from shifted states to the dissociated state than to more aligned states, indicating that the slithering-hybridization process is frequently disrupted by complete dissociation \textcolor{purple}{This is consisting with the GCGCGC sequence of Wales, right? They saw a large barrier to association from the double shifted state}. In particular, we observe that the transition probability from double-shifted 3' state to the dissociated state is 6x higher than to the neighboring single-shifted 3' state. Although there is an approximately equal probability of transitioning from the dissociated state to the hybridized and 5' shifted states, there is a 2x lower probability of moving to the 3' shifted states. These asymmetric kinetic effects may have some role in the substantially lower state populations we see above. From these observations we conclude that shifted states are important in facilitating the hybridization process, but that increasingly shifted states (in particular when shifted in the 3' direction) are more likely to dissociate than to evolve into a fully aligned hybridized state \textcolor{purple}{Based on the thermo studies of dangling ends and inert tails, it seems like the role of shifted states may depend on salt conditions, this could be an interesting future direction.}.

For all proceeding sequences, we use the same SRV and MSM lag times, number of microstates, and cross-validation procedures as in the AT-all case. Although these might not represent fully optimized hyperparameters for each sequence, they preserve Markovian properties of the system and provide adequate resolution to evaluate macrostates \textcolor{purple}{not sure if this undermines the point I make earlier about building unique microstates and macrostates for each sequence so as to optimize the model}. The number of SRV coordinates and MSM macrostates are varied based on the kinetics and thermodynamics behavior of specific sequences.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\subsection{\label{sec:Results}GC-end}

\subsubsection{\label{sec:Results}SRV-MSM construction and states}
	
Next we examine the GC-end sequence, which add GC caps to the same repetitive AT motifs. Based on SRV cross-validation (SI) we use the leading four slow modes to define our MSM basis.  These coordinates are qualitatively similar to those we studied for AT-all, where the three faster modes each represent some slithering dynamics. When we build the SRV-MSM for the GC-end sequence we observe a larger separation between the leading SRV-MSM timescale and the proceeding modes, which is shown in SI Figure \ref{fig:GC-end_dynamic}) by a distinct spectral gap. We observe a second spectral gap after the fourth slow modes along with a drop in the model vamp-2 score (SI Figure \ref{fig:srv_crossval}). The clustered macrostates show a similar distribution where shifted populations are notably lower. The 3' double-shifted state is no longer identified as a metastable state, and the total number of states is reduced to five (again one more than the total number of slow modes). Although these states appear similar to the AT-all shifted states, the presence of the C:T or G:A mismatches decreases the number of out-of-register WC pairs by two relative to corresponding AT-all states. This substantially diminishes the stability of the GC-end shifted states, resulting in lower state populations and shorter timescales.

\begin{figure}[ht!]
	\begin{center}
        \includegraphics[width=\textwidth]{Figs/figs_0804/GC-end_dynamic.png}
        \caption{(SI) GC-end SRV-MSM timescales convergence and implied timescales using a lag time of 1.2 ns. Leading slow mode correlations to all 100 intermolecular distances between oligos.}
        \label{fig:GC-end_dynamic}
	\end{center}
\end{figure}

\subsubsection{\label{sec:Results}Comparison between GC-end and AT-all shifted states}

For our analysis, we consider C:T and G:A base pairs in the GC-ends shifted states as non-interacting dangling ends such that each shifted conformation has four total dangling ends and two inert tails. Although internal base pair mismatches can cause substantial conformational distortions such as kinking, terminal mismatches have been shown to be slightly stabilizing and have a minimal effect on helical character \citep{Santalucia2004TM}, \citep{DiMichele2014EffectHybridization}. In the context of the 3spn2 model, these ends are accounted for via intra-strand base stacking and inter-strand cross-stacking interactions \citep{Hinckley2013AnHybridization} \textcolor{purple}{You may want to be more clear about the excluded volume for each of the nucleobases since this could have a large impact on the GC-ends shifted states. }. The only direct interaction between non-WC basepairs is parameterized by isotropic excluded volume potential, which is likely much more simplistic than the true mismatch interaction. Nevertheless, we find these bonds to be stabilizing enough to account for the relatively small population of conformation we observed in these shifted states (Figure \ref{fig:GC-end_in_one}). Contrary to AT-all thermodynamics, nearest neighbor calculations predict the 3' shifted states to be slightly more stable than 5' shifted states, again without taking into account entropic or enthalipic (electrostatic) effects from inert tails \textcolor{Purple}{Message here should be that inert tails, looping, other effects have more substantial contribution than two-state NN calculation alone}. Interestingly, we found that 5' single-shifted stability might be elevated by a substantial portion of conformations retaining one intact GC base pair. Visualizations reveal that the shifted oligos -- particularly in the 5' shifted state -- have a tendency to sacrifice some helical conformational entropy in a way that facilitates G:C termini bonding even when all available A:T bonds are formed out-of-register.

\begin{figure}[ht!]
	\begin{center}
        \includegraphics[width=\textwidth]{Figs/figs_0804/GC-end_in_one.PNG}
        \caption{a) Thermo value calculated from Santalucia data and from MSMs b) TICA projections of freen energies and state clustering c) and transitions probabilities between states \textcolor{Purple}{Need to fix 5' double shifted sequence, make same adjustments as for AT-all above}}
        \label{fig:GC-end_in_one}
	\end{center}
\end{figure}

\subsubsection{\label{sec:Results}Diffusion maps show sequence differences in shifted states}

To compare these single-shifted 5' prime states with the corresponding AT-all state, we employ diffusion maps built on an equal sampling of 5000 conformations from 5' shifted state of both sequences. Diffusion maps generate a low dimensional embedding based on some metric for diffusive distance and are well-suited to find subtle structural differences in temporally disconnected data \citep{Coifman2006DiffusionMaps}. We used all 100 intermolecular distances (as opposed to the 55 permutation free coordinates used to construct SRV-MSM) as our distance metric, making it easier to discern structures that form on either permutable end of the shifted conformation. This created a degenerate 2nd and 3rd diffusion modes, with nearly equal eigenvalues, differentiating between looping at the identical "top" and "bottom" of the strands. In Figure \ref{fig:GC-end_dmaps} we present the first two non-trivial diffusion map eigenfunctions and show representations of the degenerate third coordinate in the SI (Figure \ref{fig:GC-end_dmaps_full}. The first diffusion mode clearly delineates between the GC-end and AT-all shifted conformations and correlates highly with the average distance between the 3' end and its shifted complementary pair. This reveals that the mismatch C:T-pairs are never bound -- a consequence of the excluded volume interaction \textcolor{purple}{state explicitly that C and T have greater excluded volume than G and A in 3spn.2}-- whereas the AT-all pairs are mostly bound with occasional fraying indicated by small AT-all overlap in the GC-end region. \textcolor{purple}{Will include 3' shifting comparision in SI}. The second diffusion mode, which correlates highly with the average distance between 3' and 5' ends, has higher values for GC-end than AT-all. Because the GC-end termini do not bind out of register, we find that they are readily able to form stabilizing contacts despite the shifted conformation of the duplex as a whole. These "shifted-loop" bonds are shown to be uniquely stable for GC-end conformations in the 5' shifted state, and their existence in the simulations confirmed by molecular renderings of these regions. Although AT-all shifted ends tend to stay bound out-of-register, we find the second diffusion map coordinate to be indicative of inert tails that fold back onto these terminal out-of-register pairings.

\begin{figure}[ht!]
	\begin{center}
        \includegraphics[width=\textwidth]{Figs/figs_0804/GC-end_dmaps.png}
        \caption{First two diffusion map coordinates built from 10000 single-shifted 5' states, equally sampled from AT-all and GC-end. Color maps show inverse distances between out-of-register ends and complementary ends. }
        \label{fig:GC-end_dmaps}
	\end{center}
\end{figure}

\begin{figure}[ht!]
	\begin{center}
        \includegraphics[width=\textwidth]{Figs/figs_0804/GC-end_dmaps_full.PNG}
        \caption{SI fig }
        \label{fig:GC-end_dmaps_full}
	\end{center}
\end{figure}

\subsubsection{\label{sec:Results}GC-end dynamics are largely two-state}

The kinetic behavior between GC-end macrostates is also distinct compared to AT-all. We do not observe any significant flux between the double-shifted to single-shifted 5' states. While this could be due to insufficient sampling, it is likely that the double-shifted states has too few contacts to directly transition into a single-shifted state without first dissociating. This is further demonstrated by the kinetic distance between this state and the aligned hybridized state (S0 and S4, respectively) shown in TICA projections (Figure \ref{fig:GC-end_in_one}). We observe some flux between the single-shifted states and aligned hybridized state, but these events are more rare than their AT-all equivalents. This suggests that although these transitions are possible routes for hybridization/dissociation, it is substantially more common for the transition to proceed in a two-state manner. In T-jump experiments, \textcolor{purple}{In addition, there was less deviation from all-or-nothing behavior in the FTIR and 2D IR temperature series} the GC-end sequences was observed to have less deviation from the two-state dissociation model compared to oligonucleotides with GC pairs closer to the core \citep{Sanstead2016}. Futhermore, there was a distinct kinetic response -- particularly at lower temperature -- that did not clearly match the fraying processes under investigation. \citep{Sanstead2018DirectDehybridization}. Given that this signal contained some A:T and G:C character and was slower than single base pair fraying, it is possible that it represented some oligomer population shifting out-of-register as we observe in our MSMs. This cannot be verified given that the signal was recorded in a congested spectroscopic range, however this dynamic should be taken into consideration when analyzing or utilizing oligos of a similar sequence \textcolor{purple}{I think the real problem is that its unclear if IR spectroscopy (or most probes for that matter) can distinguish between fraying and shifting. Both will appear as the loss of AT and GC base pairs. The only thing I can think of is if we could see cross-peak between C and T or G and A features in a T-jump 2D IR surface. This is where spectral congestion could be a real problem.
}.

\begin{figure}[ht!]
	\begin{center}
        \includegraphics[width=\textwidth]{Figs/skeleton/shifting_distribution.PNG}
        \caption{Shows similarities between shifting modes and state memberships for AT-all and GC-ends (Not sure if this adds anything, if anything can grey out underlying free energy landscape and move this to the SI.}
        \label{fig:shifting_distributions}
	\end{center}
\end{figure}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\subsection{\label{sec:Results}GC-core}

\subsubsection{\label{sec:Results}Building SRV-MSM}

The GC-core sequence represents a departure from the dominant slithering \textcolor{purple}{Need to pick a a slither/shift term to stick with. "Slithering" has been used most in the past, but "slithered " state doesn't sound right} dynamics observed for AT-all and GC-end. Based on SRV cross-validation (SI Figure \ref{fig:srv_crossval}), we selected the first three components for our analysis. Next, we built an SRV-MSM using these first three SRV modes as a basis and proceeded along the pipeline as described above. We found that four macrostate clustering was unstable -- likely because the 3rd mode is mostly providing  information about dissociation dynamics -- so we performed PCCA+ clustering into three macrostates representing the hybridized, dissociated, and "4bp-frayed" states. Again, we found transition probabilities between states and visualized representative molecular renderings. As we observed from the 2nd SRV mode analysis above, the 4bp-frayed meta-stable state is solely composed of trajectories where both G:C core base pairs are bound and one of the adjacent A:T bonds are not. Previous work suggests that once key contacts are made, the zippering mechanism ensures that the helix will quickly form outward \citep{Romano2013DNADependence}, \citep{Yin2011KineticsHybridization}. Our results indicate, however, that the relative instability of AT bonds compared the the GC-core can interrupt this process and form a longer lived frayed metastable states. This occurs during the dissociation process as well, where one half of the A:T base contacts are entirely broken for a substantial period of time before the full dissociation event occurs. We observe these events to happen with equal probability on either permutable side of the helix. The MSM transition probabilities suggests that this is key intermediate state for both the hybridization and dissociation processes, however the pathways differ between directions. We find that the hybridized state has a 5x higher probability of transitioning into the frayed state within the lag time compared to transitions from the dissociated state. This is expected considering that this state is more accessible from an already bound helix. Moreover, once oligos are in this state, they are over 10x more likely to return to the hybridized state than to fully dissociate. Thus once a dissociated to 4bp-frayed transition has occurred, it is a likely to proceed into a fully hybridized conformation. On the other hand, transitions from the hybridized to 4bp-frayed state are much more frequent and are unlikely to proceed to a fully dissociated state.

\begin{figure}[ht!]
	\begin{center}
        \includegraphics[width=\textwidth]{Figs/figs_0804/GC-core_thermo.PNG}
        \caption{Full MSM output for GC-core}
        \label{fig:GC-core_thermo}
	\end{center}
\end{figure}

\subsubsection{\label{sec:Results}Diffusion maps show ensemble of frayed states}

To further examine this 4bp-frayed state, we build diffusion maps using 10000 frames sampled from the macrostate. This time, we set out distance metric to the same permutation-free coordinates we used to build the SRV-MSMs. Again, we were able to identify a combination of physical coordinates that closely correlated to the first two non-trivial diffusion modes. We found that a larger distance between the third and fourth A:T pairs increased the likelihood that a configuration was clustered into the meta-stable macrostate. This distance also correlated closely with the second diffusion map mode. Interestingly, we observed that the first non-trivial mode -- the feature that describes the most structural diversity in the system -- corresponds to difference in "overlap" distance between adjacent A or T bases and the GC core. In these conformations, one of the strands maintains some helical character while the other twist out of place, resulting in WC bonds being obstructed by the oligo backbone. These states represent another potential way in which the hybridization process (or helix reformation) can be kinetically frustrated. We observed this mode to be mostly symmetric, however there is tendency for the 3'T end to fray farther out of place relative to its 5'A counterpart \textcolor{purple}{This seems like a very small difference. Could be due to undersampling or another excluded volume effect}.

\begin{figure}[ht!]
	\begin{center}
        \includegraphics[width=\textwidth]{Figs/figs_0804/GC-core_dmaps.png}
        \caption{Plotting the first two non-trivial diffusion map eigenfunctions. Color maps show the probability that conformations are clustered into the frayed macrostate, maximum distance between 4th AT basepairs, and the the overlap between adjacent AT basepairs and the GC core.}
        \label{fig:GC-core_dmaps}
	\end{center}
\end{figure}

\subsubsection{\label{sec:Results}SRV correlations to physical coordinates?}

Having constructed and interpreted our SRV-MSM, we revisited our original SRV basis to analyze what physical correlations could be interpreted from the model. We found these GC-core modes to be of particular interest as they reveal the hierarchical nature of the dynamical encoding. We examined a collection of 1000-frame "trimmed" trajectories centered on both hybridization or dissociation events. For each trajectory, we compared the first three SRV coordinates with a corresponding collective variables with which they showed strong correlation. Two representative trajectories are shown in figure: (\ref{fig:GC-core_tracking_modes}). Complementary G:C pairs are the best indicators for a hybridization/dissociation event, and we see a sharp change in the first SRV mode (SRV1) as these bonds form or break. The second slow mode (SRV2) is most active when G:C pairs are bound but the adjacent AT pairs are not. There is a small signal for fraying at the outer base pairs, but the mode overwhelmingly learns about these neighboring A:T/G:C bonds. This behavior reflects the above-mentioned kinetic trap between core binding and the fully hybridized state. SRV3 is most active during dissociation, and seems to track closely with the average distance between all complementary base pairs. We attribute this to the SRV learning about the diffusive motions of the two body system. In addition to picking up on dissociation behavior, the third mode peaks when the oligos are close together but configured in such a way that is not amenable hybridization. These misaligned conformations include inverse contacts where 5'/5' and 3'/3' ends meet and looped conformations where one strand is folded in on itself and preventing satisfactory WC contacts. 

Despite the strong qualitative trends we observe between physical coordinates and leading SRV modes, we only find high Pearson correlations for SRV1. For the next two SRV modes, the sign of their correlation switches depending on whether the oligos are in the hybridized or dissociated state. This shows that these modes are providing support on top of the first mode -- which determines hybridization vs. dissociation -- and thus can display very different behaviors in either state. With respect to our SRV-MSM macroscates, we found that SRV2 was "turned on" in states 0 and 1 -- corresponding to the intact helix and frayed state, respectively -- and SRV3 was turned on in the dissociated state 2. Accordingly, we recalculated Pearson correlations between each SRV mode and all distances in states where the modes are active. Figure \label{fig:GC-core_srv_correlations} shows the highest correlation between SRV2 and inner A:T pairs, weak correlation with outer A:T pairs, and an inverse correlation with 5'/5' and 3'/3' pairs which tend to approach each other during 4bp-fraying. We also observed a highly symmetric correlation between SRV3 and central base pairs distances, indicative of overall diffusive behavior. Taken together, these analyses reveal how the SRV learns and represents the dynamical space, leading to the MSM results we observe above.


\begin{figure}[ht!]
	\begin{center}
        \includegraphics[width=\textwidth]{Figs/figs_0804/GC-core_tracking_modes.png}
        \caption{Show how SRV coordinates correspond to physical features over time and during sample dissociation and hybridization events. Should probably go to SI if we also include GC-core diffusion maps \textcolor{purple}{These single trajectory figures have proven useful for visualizing SRV dynamics, could be employed for GC-mix comparisons as well}}
        \label{fig:GC-core_tracking_modes}
	\end{center}
\end{figure}    

\begin{figure}[ht!]
	\begin{center}
        \includegraphics[width=\textwidth]{Figs/figs_0804/GC-core_srv_correlations_by_state.png}
        \caption{Shows Pearson correlations between leading SRV modes and all 100 intermolecular distances (needs labels for slow modes)}
        \label{fig:GC-core_srv_correlations}
	\end{center}
\end{figure}

\subsubsection{\label{sec:Results}Comparison to Experiment?}

When examining these same four sequences using T-jump IR and 2D IR spectroscopy, Sandstead et al. found that the GC-core had the highest deviation from two-state behavior during dissociation \citep{Sanstead2016}. As their lattice model did not consider previously mentioned shifted states, this intermediate state was defined by a high degree of fraying about the central core. While 1-2 base pair fraying was commonly observed for GC-mix and AT-all as well, lattice model predictions showed that GC-core had substantially more frayed base pairs \citep{Phys2019}. Variable T-jump measurements and Smoluchowski simulations on model 1D free energy landscapes showed that showed that AT termini fraying was an effectively barrierless process characterized by rapid inter-conversion between all accessible frayed states \citep{Sanstead2018DirectDehybridization}. We see the same rapid fraying in simulation data -- which is too fast to be attributed to a converged SRV mode -- however we stipulate that this inter-conversion first relies on the formation of the the A:T bond nearest to the GC center. Our diffusion map analysis further shows how the ensemble of "macrofrayed trajectories" impede helix formation. Although this process occurs much slower than single A:T base bonding and breaking, it may be difficult to experimentally discern from the overall hybridization process which contains both G:C and A:T character and occurs on a similar timescale.

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\subsection{\label{sec:Results}GC-mix}

The GC-mix sequence shows a similar implied timescale distribution to GC-core, however we no longer see a converged slow mode corresponding to frayed behavior. Although we observe substantial fraying of the two AT termini in the simulation data, these dynamics are too fast to converge as distinct frayed modes. Instead, we observed two modes converge, corresponding to the association/dissociation dynamic and diffusive behavior the strands while dissociated. These correlate closely with the first and third GC-core SRV modes. Although we built our SRV-MSM using these two coordinates, we again were unable to form a stable third state along the second coordinate. As such, we designated this transitions as an effectively two-state \textcolor{purple}{I think there needs to be better clarity for the difference between "two-state" and "all-or-nothing".Your model gives only 2 clustered states, but the faster fraying motions still indicate that the dissociation is not all-or-nothing} process with no long-lived or kinetically relevant metastable states. Previous work has attributed AT-all termini fraying prior to dissociation as a deviation from two-state behavior \citep{Sanstead2016}. Here we consider these AT-frayed states to be part of the overall hybridization state. While AT-termini fraying is surely a prerequisite for dissociation, we find these states to be so common and fleeting that very few evolve initiate dissociation. Furthermore, they do not fundamentally disrupt the helix in such a way that its re-formation is kinetically inhibited by the intermediate structures we present for GC-core.

\subsubsection{\label{sec:Results}Definition of two-state and nucleation}

In the simulation data, we do observe sustained regions of partial base pairing during hybridization. These states reflect initial contact that form between base pairs prior to a fast zippering mechanism that binds the oligos into a fully hybridized state. Qualitatively, these observations match well with the three-state assumption for hybridization that proceeds through a "nucleation" state of around 3 base pairs bound \citep{Yin2011KineticsHybridization, Porschke1971CooperativeTransition} \textcolor{purple}{It may be a good idea to briefly discuss which sequences follow a nucleation-zipper model which show deviations. The nucleation-zipper is the most kinetic model for interpreting DNA hybridization, but it is probably a serious oversimplification in many cases!}. Our featurization method, however, was designed to elucidate sequence-dependent -- thus location-dependent -- dynamics, and does not naturally group these kinds of state. This limitation also does not allow us to pick up on proceeding zippering mechanisms, as these would each appear as separate dynamics per the independent nucleation site and need to be sampled accordingly.

Other ways to investigate GC-mix:
\begin{enumerate}
	\item Can try building simplified MSM here: (use number of base pairs bound as features)
    \item Timescale analysis shows that GC-mix is reversible two-state process. We note that it takes about 40 percent longer for a fully hybridized oligomer to fully dissociate (once > 2 bp are borken) for GC-core compared to GC-mix. 
	\item Given that GC-mix and GC-core are run at very similar temperature, we would expect the extra basepairs broken during GC-mix dissociation process to slow dissociation, however in the opposite effect was observed in the slow mode data
	\item Evidence of fraying correlations in higher order modes, but these are not consistent and their inclusion does not generate a metastable third mode
	\item Show 2nd slow mode correlations in SI plus some configuration snapshots
	\item Discuss misaligned configurations here instead of in the GC-core section
\end{enumerate} 
 
\subsection{\label{sec:Results}Project results onto shared basis}

Here I've constructed a shared basis from physical coordinates informed by the AT-all SRV2 and GC-core SRV2 (Figure \ref{fig:all_seq_physical_cvs_labels}) and TICA coordinates learned on a combination of all four sequence (Figure \ref{fig:all_seq_tica_cvs}). I'm leaning towards presenting the first in the main text and including the other in the SI, but I'm not sure how clear either of these are.

\begin{figure}[ht!]
	\begin{center}
        \includegraphics[width=\textwidth]{Figs/figs_0804/all_seq_physical_cvs_labels.PNG}
        \caption{All sequences and metastable macrostates plotted on frayed/shifted collective variable axes, Not sure how helpful these axes drawings are -- also need to switch x and y here.}
        \label{fig:all_seq_physical_cvs_labels}
	\end{center}
\end{figure}

\begin{figure}[ht!]
	\begin{center}
        \includegraphics[width=\textwidth]{Figs/figs_0804/all_seq_tica_cvs.PNG}
        \caption{All sequences plotted on combined TICA axes. Can add similar macrostate distribution as shown in the above figure.}
        \label{fig:all_seq_tica_cvs}
	\end{center}
\end{figure}


\subsubsection{\label{sec:Results}Limitations} 

\begin{enumerate}
	\item Does not explicitly pick up on nucleation/zippering events
	\item Coarse grain-models makes it difficult to compare timescales. Furthermore our implied timescales function more as an average frequency of a process rather than a value that can be directly compared to experiment. 
	\item Limited by memory (number of frames) required to capture equilibrium trajectories at a save rate that enabled adequate resolution for the dynamics of interest.
	
\end{enumerate}


\section{\label{sec:conc}Conclusion}
\subsection{\label{sec:Results}Will look a lot like the last paragraph of the intro / the abstract}


\begin{comment}
Classic model has hybridization forming in three distinct step:
A nucleation step involves bringing two random- coil ssDNAs together to form an intermediate state, in which a few base pairs have formed, and this intermediate pro- ceeds further to form the fully hybridized dsDNA. 

- 

- A smoother free energy surface minimizes the time spent in these intermediate states, and our featurization method 

- not capturing zippering
One limititation of our appraoch that is that processes that are. For example, there are eight 3-bp sites on each oligo from which a zippering action coud nucleated. Because these are all unique sites (and zippering dynamics could vary at each site) it would be difficult to properly sample and interpret the unique corresponding mode. Furthermore these zippering dynamics occur significantly faster than slithering or GC-core fraying and cannot be captured via our save rate.

%%%%%% intro notes %%%%%%%
\citep{Sanstead2018DirectDehybridization} Good intro with many repesimportance of dynamics "rich dynamics even for short oligos"

%%%%%% Other MSM and timescales approaches  %%%%%%%%%%%

McGibbon, R. T., & Pande, V. S. (2015). Variational cross-validation of slow dynamical modes in molecular kinetics : Importance of not generating too many microstates and employting cross-validation to avoid artificially high scores or artificially slow dynamics. Both of these lead to models fitting statistical noise which is the root of boosted scores.

Pinamonti, G., Paul, F., Rodriguez, A., & Bussi, G. (n.d.). analyzed with core-set Markov state models, 43: Good ideas on core-based clustering. Assigning microstates based on physical characteristics is interesting but didn't seem to work for me in practice (could try doing this based on sim.log data with number of base pairs bound or some energy metric). Lot's of notes on this paper regarding comparisons to timescales. Good notes on higher resolution fraying interactions that we likely can't resolve.

Sua, E., Adelman, J. L., & Zuckerman, D. M. (2016). Accurate Estimation of Protein Folding and Unfolding Times : Beyond Markov State Models, 1. https://doi.org/10.1021/acs.jctc.6b00339:
More accurate ways to adapt MFPT into a rate

Refs for DNA fraying:
\citep{Hagan2003AtomisticDNA} Atomistic simulations for biosensign app (can include in intro). Use TPS to focus on kinetic pathway by which singl base pair bind/unbind. Focus on end base of 3-bp oligomer in explicit solvent. Finds order parameters tracking base pair binding as well as intra-strand stacking. (energy and distance for each). 

\citep{Wong2008TheSimulations} Three step melting process: untwisting most important,

6K.-Y. Wong and B. M. Pettitt, “The pathway of oligomeric DNA melting investigated by molecular dynamics simulations,” Biophys. J. 95, 5618– 5626 (2008).
37A. Perez and M. Orozco, “Real-time atomistic description of DNA unfold- ing,” Angew. Chem. 49, 4805–4808 (2010).
38M. F. Hagan, A. R. Dinner, D. Chandler, and A. K. Chakraborty, “Atomistic understanding of kinetic pathways for single base-pair binding and unbinding

\end{comment}


\clearpage
\newpage

%\bibliography{references}
\bibliography{refs_mendeley2}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% The "tocentry" environment can be used to create an entry for the
%% graphical table of contents.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\clearpage


\end{document}
