%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% This is a (brief) model paper using the achemso class
%% The document class accepts keyval options, which should include
%% the target journal and optionally the manuscript type.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\documentclass[journal=jpcbfk,manuscript=article]{achemso}

%\documentclass[english,aps,preprint,pre,floatfix,nofootinbib,showpacs,showkeys]{revtex4-1}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Place any additional packages needed here.  Only include packages
%% which are essential, to avoid problems later. Do NOT use any
%% packages which require e-TeX (for example etoolbox): the e-TeX
%% extensions are not currently available on the ACS conversion
%% servers.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\usepackage[version=3]{mhchem} % Formula subscripts using \ce{}

\SectionNumbersOn

%include hyperlinks to figures
%\documentclass{article}
\usepackage{graphicx}
\usepackage[%  
    colorlinks=true,
    pdfborder={0 0 0},
    linkcolor=blue,
    citecolor=black
]{hyperref}

\usepackage{amsmath}
\usepackage{amssymb}
\usepackage{color}
\usepackage{CJK}
\usepackage{framed}

%add refs to SI
%\usepackage{xr}
%\externaldocument{SI}

%\usepackage[hidelinks]{hyperref}
%\usepackage{algorithm}
%\usepackage{algpseudocode}
%\algdef{SE}[DOWHILE]{Do}{doWhile}{\algorithmicdo}[1]{\algorithmicwhile\ #1}

\usepackage[scaled=0.85]{beramono}
\usepackage[T1]{fontenc}
\usepackage{changepage}
\usepackage{mathrsfs}
\usepackage{comment}
\usepackage[normalem]{ulem}
\usepackage{algorithm}
\usepackage{subfig}
\usepackage{algpseudocode}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% If issues arise when submitting your manuscript, you may want to
%% un-comment the next line.  This provides information on the
%% version of every file you have used.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%\listfiles

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Place any additional macros here.  Please use \newcommand* where
%% possible, and avoid layout-changing macros (which are not used
%% when typesetting).
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\makeatletter
    \setlength\@fptop{0\p@}
\makeatother

\newcommand*\mycommand[1]{\texttt{\emph{#1}}}

\newcommand{\state}[1]{$\mathcal{S}_{#1}$}
\newcommand{\Expect}[1]{\mathrm{E}\left[#1\right]}
\newcommand{\norm}[1]{\left\lVert#1\right\rVert}


\newcommand*{\rood}[1]{{\color{red}{#1}}}
\newcommand*{\bood}[1]{{\color{blue}{#1}}}
\newcommand*{\blauw}[1]{\textcolor{blue}{#1}}
\newcommand*{\groen}[1]{\textcolor{green}{#1}}

\newcommand*{\addref}[1]{\textcolor{red}{\{ADD REF: #1\}}}
\newcommand*{\noter}[1]{\textcolor{red}{[[#1]]}}		% notes 
\newcommand*{\noteb}[1]{\textcolor{blue}{[[#1]]}}		% notes 
%\newcommand*{\noter}[1]{}					% notes off

%\usepackage[draft]{todonotes}   % notes shown
\usepackage[disable]{todonotes} % notes hidden



%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Meta-data block
%% ---------------
%% Each author should be given as a separate \author command.
%%
%% Corresponding authors should have an e-mail given after the author
%% name as an \email command. Phone and fax numbers can be given
%% using \phone and \fax, respectively; this information is optional.
%%
%% The affiliation of authors is given after the authors; each
%% \affiliation command applies to all preceding authors not already
%% assigned an affiliation.
%%
%% The affiliation takes an option argument for the short name.  This
%% will typically be something like "University of Somewhere".
%%
%% The \altaffiliation macro should be used for new address, etc.
%% On the other hand, \alsoaffiliation is used on a per author basis
%% when authors are associated with multiple institutions.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\author{Michael S. Jones}
\affiliation{%
  Pritzker School of Molecular Engineering, %
  The University of Chicago, %
  929 East 57th Street, Chicago, Illinois 60637, United States%
}

\author{Brennan Ashwood}
\affiliation{%
  Department of Chemistry, Institute for Biophysical Dynamics, and James Franck Institute, %
  The University of Chicago, %
  929 East 57th Street, Chicago, Illinois 60637, United States%
}

\author{Andrei Tokmakoff}
\affiliation{%
  Department of Chemistry, Institute for Biophysical Dynamics, and James Franck Institute, %
  The University of Chicago, %
  929 East 57th Street, Chicago, Illinois 60637, United States%
}

\author{Andrew L. Ferguson}
\email{andrewferguson@uchicago.edu}
\affiliation{%
  Pritzker School of Molecular Engineering, %
  The University of Chicago, %
  929 East 57th Street, Chicago, Illinois 60637, United States%
}


%\author{Kay T. Finally}
%\affiliation[Unknown University]
%{Department of Chemistry, Unknown University, Unknown Town}
%\alsoaffiliation[Second University]
%{Department of Chemistry, Second University, Nearby Town}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% The document title should be given as usual. Some journals require
%% a running title from the author: this should be supplied as an
%% optional argument to \title.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\title[]{Determining sequence-dependent DNA oligonucleotide hybridization and dehybridization mechanisms using coarse-grained molecular simulation, Markov state models, and infrared spectroscopy}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Some journals require a list of abbreviations or keywords to be
%% supplied. These should be set up here, and will be printed after
%% the title and author information, if needed.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%\abbreviations{IR -- infrared, NMR -- nuclear magnetic resonance, UV -- ultraviolet}
%\keywords{Takens}

\begin{document}
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% The manuscript does not need to include \maketitle, which is
%% executed automatically.  The document should begin with an
%% abstract, if appropriate.  If one is given and should not be, the
%% contents will be gobbled.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\newpage

\begin{abstract} % 250 words max for JACS

\noindent A robust understanding of the sequence-dependent thermodynamics of DNA hybridization has enabled rapid advances in DNA nanotechnology. A fundamental understanding of the sequence-dependent kinetics and mechanisms of hybridization and dehybridization remains comparatively underdeveloped. In this work, we establish new understanding of the sequence-dependent hybridization/dehybridization kinetics and mechanism within a family of self-complementary pairs of 10-mer DNA oligomers by integrating coarse-grained molecular simulation, machine learning of the slow dynamical pathways, data-driven inference of long-time kinetic models, and experimental temperature-jump infrared spectroscopy. For a repetitive ATATATATAT sequence \noteb{not sure if we want to use 5$^\prime$-ATATATATAT-3$^\prime$ notation here as well?}, we resolve a rugged dynamical landscape comprising multiple metsatable states, numerous competing hybridization/dehybridization pathways, and a spectrum of dynamical relaxations. Introduction of a G:C pair at the terminus (GATATATATC) or center (ATATGCATAT) of the sequence reduces the ruggedness of the dynamics landscape by eliminating a number of metastable states and reducing the number of competing dynamical pathways. Only by introducing a G:C pair midway between the terminus and the center to maximally disrupt the repetitive nature of the sequence (ATGATATCAT) do we recover a canonical two-state model of hybridization/dehybridization over a smooth landscape with no intermediate metastable states. Our results establish new understanding of the dynamical richness of sequence-dependent kinetics and mechanisms of DNA hybridization/dehybridization, present a molecular basis with which to understand experimental temperature jump data, and furnish foundational design rules by which to rationally engineer the kinetics and pathways of DNA association and dissociation in burgeoning DNA nanotechnology applications.

%\noindent \url{https://www.overleaf.com/project/5e9e5110c524b8000192c548}

\end{abstract}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Start the main part of the manuscript here.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\newpage

\section{\label{sec:intro}Introduction}

Over the last couple of decades, DNA has proven to be much more than a vessel for genetic information. From sensing to computing to directed self-assembly, the programmable and predictable nature of DNA has unlocked numerous unforeseen nanotechnology applications.\citep{Seeman2017DNANanotechnology, Adleman1994MolecularProblems, Rothemund2006FoldingPatterns, Gu2010ALine} Recently, single molecule localization techniques have exploited the rapid and transient binding of short DNA oligomers in order to achieve super-resolution microscopy and optical multiplexing. \cite{Schnitzbauer2017Super-resolutionDNA-PAINT, Shah2019, Strauss2020UpDNA-PAINT} Predictive understanding of the sequence-dependent thermodynamics of DNA hybridization/dehybridization -- the assembly/disassembly of a DNA duplex from two single strands -- has underpinned the rational design of DNA oligomer sequences for nanotechnological applications, where sequence-dependent nearest-neighbor models can accurately account for mismatched pairs, dangling ends, and other non-native bonding effects.\citep{SantaLucia1998AThermodynamics, Santalucia2004TM} Secondary DNA structures such as hairpins and G-quadruplexes have also been studied in depth and leveraged for nanotechnology applications.\citep{Tsukanov2013DetailedOrigami, Mosayebi2014TheFormation, Mergny2019DNANanotechnology} Predictive models of the \textit{dynamical}, as opposed to purely thermodynamical, behaviors of DNA have become increasingly important in developing technologies such as DNA-PAINT (DNA Points Accumulation For Imaging In Nanoscale Topography), but developments in dynamic DNA nanotechnologies have outpaced our understanding of the dynamics themselves. \citep{Deluca2020DynamicDevices, Cordes2010SensingTransfer, Naimark2020DNADevelopment} Many experimental and computational studies have investigated DNA dynamical phenomena from the picosecond to millisecond range.\citep{Yin2011KineticsHybridization, Xiao2019, Hinckley2014Coarse-grainedEffects, Sanstead2016, Porschke1971CooperativeTransition} Kinetic models have been developed for particular DNA processes such as toehold exchanges and optical barcoding \citep{Zhang2009ControlExchange, Shah2019ProgrammingFingerprinting} and supervised machine learning techniques have been combined with experimental measurements to predict the on/off rates as a function of sequence. \citep{Schickinger2018TetheredHelices, Zhang2018PredictingSequence, Shah2019}. A comprehensive understanding of the full dynamical landscape of hybridization/dehybridization accounting for the sequence-dependent metastable states and association/dissociation pathways remains lacking and fundamental questions remain unresolved. For example, it remains unclear the extent to which hybridization of short DNA oligomers largely proceeds in a conventionally assumed ``all-or-nothing'' fashion or if long-lived metastable states facilitate the transition.\citep{Araque2016LatticeCooperativity, Sikora2013ModelingIntermediates, Wyer2014KineticsAT-tracts, Sanstead2016, Sanstead2018DirectDehybridization, Xiao2019} Out-of-register ``shifted'' base paired structures \citep{Phys2014, Maciejczyk2014DNAModel, Araque2016LatticeCooperativity, Xiao2019} and frayed structures \citep{Zgarbova2014BaseRNA, Nonin1995TerminalFraying, Nikolova2012ProbingSimulations, Andreatta2006UltrafastHelix} stand as candidates for metastable states with the potential to mediate substantial deviations from all-or-nothing behavior, but the extent to which these states are kinetically relevant is difficult to determine experimentally and is likely to be highly sequence-dependent. The development of predictive models and design rules with which to engineer DNA strands with tailored hybridization/dehybridization kinetics and pathways is vital to advancing rational design of DNA strands for nanotechnology applications and is also of importance in understanding fundamental biological processes such as transcription and gene regulation.

Our understanding of hybridization dynamics has been built from decades of experiments -- such as temperature-jump, salt-jump, pH-jump, and other perturbative methods -- that rapidly stimulate DNA and monitor relaxation to a new equilibrium.\citep{Morrison1993SensitiveSolution, Wetmur1968KineticsDNA, Craig1971RelaxationOligonucleotides, Porschke1973ThermodynamicsPairs, Williams1989LaserDGCATGC, Narayanan2012ExploringMixing, Chen2007InfluenceHybridization, Sanstead2018DirectDehybridization} More recently, single molecule diffusion and tethered multifluorophore assays have facilitated equilibrium analysis, but these results can be hampered by slow data collection rates and fluorescent tags effects on strand dynamics, particularly for shorter oligomers.\citep{Liu20173DSolution,  Schickinger2018TetheredHelices, Chen2008Base-by-baseSpectroscopy, Dupuis2013Single-moleculeHelices} A number of computational modeling approaches have also been employed to provide molecular-level resolution of hybridization. Simplified lattice models can recapitulate the essential aspects of the hybridization pathways but lack the realism of continuous space representations \citep{Araque2016LatticeCooperativity, Phys2019}. The long time scales associated with hybridization/dehybridization events place them outside the reach of unbiased all-atom molecular dynamics simulations,\citep{Phys2014} but they can be observed by employing enhanced sampling techniques such as metadynamics, umbrella sampling,  transition path sampling, and forward flux sampling,\citep{Hinckley2013AnHybridization, Schmitt2013ExploringSurface, Sambriski2009, Sambriski2009SequencePathways, Hoefert2011MolecularOligonucleotides,Romano2013DNADependence, Hinckley2014Coarse-grainedEffects} \noteb{these references correspond to CG enhanced sampling studies, I can refer to enhanced sampling on AA here instead (\citep{Piana2007AtomisticTransition, Zerze2021ThermodynamicsSimulations}) but I think the CG studies should be mentioned since not much has been done for AA hybridization} or by using elevated temperature or denaturing solvent concentrations to induce one-way dissociation events.\citep{Wong2008TheSimulations, Perez2010Real-timeUnfolding} The effect of the applied bias upon the thermodynamics can be rigorously corrected for using standard reweighting techniques, but approaches to rigorously correct the kinetics, particularly under the conditions of high bias necessary for good sampling, are in their infancy.~\citep{Prinz2011OptimalDynamics, Chodera2011DynamicalTemperatures, Stelzl2017DynamicSimulations, Donati2017GirsanovModels, Donati2018GirsanovSimulations, Quer2018ANCOORDINATES} A number of coarse-grained DNA force fields have been developed that enable direct observation of these events over microsecond time scales via unbiased coarse-grained molecular dynamics (CGMD) simulations, \citep{Romano2013DNADependence, Hinckley2013AnHybridization, Maciejczyk2014DNAModel, Markegard2015, Dans2016MultiscaleDNA} which, up to a speedup factor associated with the smoothing of the underlying free energy landscape inherent to the coarse-graining procedure, can preserve a faithful model of the unbiased dynamics and associated pathways. These models have previously been used to study biological phenomena such as nucleosome dynamics\citep{Lequieu2016Tension-dependentUnwrapping, Lequieu2017InSliding} and transcription factor binding \cite{Terakawa2015P53Simulations, Tan2018DynamicData} as well as nanotechnology applications such as strand displacements \citep{Srinivas2013OnDisplacement, Haley2020DesignDisplacement} and DNA origami structure \citep{Snodin2019Coarse-grainedOrigami, Doye2020TheOrigami}.

In this work, we study a family of self-complementary pairs of 10-mer DNA oligomers using coarse-grained molecular simulation, machine learning, and data-driven inference of long-time kinetic models to establish new understanding of the influence of sequence upon hybridization/dehybridization kinetics and mechanisms. This family of oligomers -- 5$^\prime$-ATATATATAT-3$^\prime$ (AT-all), 5$^\prime$-GATATATATC-3$^\prime$ (GC-end), 5$^\prime$-ATATGCATAT-3$^\prime$ (GC-core), and 5$^\prime$-ATGATATCAT-3$^\prime$ (GC-mix) -- was designed to probe the influence of the placement of a G:C pair within an otherwise repetitive A:T sequence and has been the subject of our prior experimental investigations using temperature-jump infrared spectroscopy and simple lattice models. \citep{Sanstead2016} We validate the new computational models of hybridization/dehybridization dynamics developed in this work against new and existing experimental data and reinterpret our prior experimental observations in light of the new computational understanding. Consistent with previous studies, \citep{Phys2014,Romano2013DNADependence,Araque2016LatticeCooperativity} we find the degree of repetitiveness in the sequence -- and therefore the kinetic accessibility and thermodynamic stability of non-native states -- leads to richer dynamics populated by a diversity of long-lived metastable states. Our data-driven modeling and analysis rigorously quantifies these behaviors and furnishes accurate predictive models of the hybridization/dehybridization rates, dynamical pathways, and metastable states. Specifically, we demonstrate that disrupting repetitive stretches of A:T bases by placement of interrupting G:C base pairs enables us to tune the landscape from rich six-state to simple two-state ``all-or-nothing'' behavior, and the specific location of the interrupting pair can be used to modulate the stability of long-lived frayed states that facilitate the hybridization/dehybridization process. Analysis of the configurational ensemble within each metastable state reveals the conformational heterogeneity and dynamical interconversions proceeding on time scales that lie below those resolvable within the long-time kinetic model. Taken together, our analyses establish new molecular-level understanding of the sequence-dependent kinetics and pathways through quantitative predictive models for the long-time system dynamics, resolution of the dynamical folding pathways and metastable states, and elementary design rules with which to sculpt and qualitatively alter the dynamical behaviors of the system. We anticipate that this new foundational understanding, and the extension of our approach to more extensive families of DNA sequences, can guide the rational design of DNA oligomers with tailored kinetic properties engineered for DNA nanotechnology applications such as DNA-PAINT. \citep{Shah2019, Strauss2020UpDNA-PAINT}


% example text from Andy's conclusions:

% Correlation of the data-driven kinetic order parameters extracted by SRVs provides new understanding of the sequence-dependent kinetic mechanisms by which oligomers hybridize and reveal these to be richer than classical base-pair shifting and “all-or-nothing” behaviors.

% original last paragraph
%By evaluating equilibrium trajectories we can study the relevance of metastable states during both the hybridization and dissociation process. Because SRVs generate an optimized low dimensional basis, we show that higher resolution MSMs (measured by a reduction in the required lag time) are accessible. Additionally, we compare slow dynamical modes and metastable states between sequence-specific SRV-MSMs. Within these metastable dynamical states, we leverage diffusion maps to analyze the diversity of structures whose inter-conversion rate are too fast to produce unique slow modes. Finally, we run higher temperatures simulations to investigate the temperature-dependent nature of some experimentally relevant dynamical responses. Taken together, our analysis reflects similar results to previous computational and experimental DNA work, while elucidating new insights into sequence-dependent dynamics, metastable structures, and relative time scales. 

%These studies have fueled increasingly complex structural DNA technologies, but developments in dynamic DNA nanotechnologies have outpaced our understanding of the dynamics themselves \citep{Deluca2020DynamicDevices}. 

%Still, rich dynamics have been observed short nucleotides \citep{Sanstead2018DirectDehybridization} and the sequence-dependent effects on these dynamics are not well understood.

% include Tokmakoff conclusions in the intro -- sequence-dependent deviations from two-state behavior. The slow response was attributed to complete strand dissociation, while the fast response described base fraying. We perform longer equilibrium sampling near Tm to get a sense of sequence equilibrium behavior, as well shorter simulations at a range of higher temperatures to replicate T-jump data. Taken together, our analysis reflects similar results to previous computational and experimental DNA work, while elucidating new insights into sequence-dependent dynamics, metastable structures, and relative time scales. 



\section{\label{sec:methods}Methods}

\subsection{Computational Methods}

\subsubsection{Molecular dynamics simulations}

We performed molecular dynamics simulations of four 10-base self-complementary double-stranded DNA sequences that we have previously studied by ultra-fast temperature-jump infrared spectroscopy \citep{Sanstead2016}: 5$^\prime$-ATATATATAT-3$^\prime$ (AT-all), 5$^\prime$-GATATATATC3-3$^\prime$ (GC-end), 5$^\prime$-ATATGCATAT-3$^\prime$ (GC-core), and  5$^\prime$-ATGATATCAT-3$^\prime$ (GC-mix). We modeled the DNA sequences using the coarse-grained 3-Site-Per-Nucleotide v2 (3SPN.2) that uses three spherical beads to represent the phosphate, deoxyribose sugar, and nitrogenous base of each nucleotide and employs anisotropic interaction potentials to accurately treat intra-strand base-stacking, inter-strand cross-stacking, and base pairing. \citep{Hinckley2013AnHybridization} The model was parameterized against experimental data on bond lengths, bend angles, torsional angles, base step energies, and base stacking free energies, and reliably reproduces the structure, melting temperatures, persistence lengths, and sequence, salt, concentration, and temperature effects on duplex formation. \citep{Hinckley2013AnHybridization} The model enables access to millisecond time scales and has been has widely-adopted to study numerous phenomenon including DNA packing in viral capsids, protein-DNA binding, and nucleosome unwrapping.\citep{Cordoba2017AIons, Lu2020OpenAWSEMSummary, Lequieu2016Tension-dependentUnwrapping} Although the 3SPN.2 model was not directly parameterized against dynamical experimental data, we will show below that the predicted sequence-dependent kinetics and relaxation are, within a corrective scaling factor, in very good agreement with observed experimental trends.

All calculations were performed using the LAMMPS simulation package (\url{http://lammps.sandia.gov}) in accordance with best practices for the 3SPN.2 model. \citep{Plimpton1997Short-RangeDynamics} A single pair of self-complementary sequences were placed in a cubic periodic box with side length 7.8 nm corresponding to a single-strand concentration of 7 mol/L. Solvent effects were modeled implicitly by employing Langevin dynamics \citep{DunwegB.Paul1991BrownianNumbers.} with an experimentally motivated per-site friction coefficient of $\xi$ = 9.94 $\times$ 10$^{-11}$ m$^2$/s. \citep{Nkodo2001DiffusionElectrophoresis, Hinckley2013AnHybridization} We specified a 240 mM implicit salt concentration and treated electrostatic interactions using the Debye-H{\"u}ckel with a 5 nm cutoff radius. \citep{Debye1923ZurElektrolyte} Simulations were performed in the NVT ensemble employing a Langevin thermostat. \citep{Schneider1978Molecular-dynamicsTransitions}. Each sequence was simulated at its melting temperature -- AT-all: 309 K, GC-end: 317 K, GC-core: 324 K, GC-mix: 324 K -- in order to maximize the number of spontaneous transitions between dissociated and hybridized states. Melting temperatures for each sequence were determined empirically by comparing the ratio of hybridized to dissociated populations over a 10 K temperature ramp centered on the Nearest Neighbors model predicted melting temperature. The temperature at which the hybridized and dissociated populations were closest was selected. The Langevin equations of motion were integrated using the scheme of Bussi and Parrinello \citep{Bussi2007AccurateDynamics} with a 20 fs integration time step. We performed 40 independent simulations for each of the four sequences with half of the runs initialized from the hybridized state and half from the dissociated state. The initial hybridized state was set, according to 3SPN.2 documentation, to the coordinates of Arnott et al.\ ~\citep{Arnott1976CRCBiology} The dissociated state was generated by shifting one oligomer from the hybridized state by 1 nm in the x, y, and z directions. The magnitude of this shift did not affect the results so long as all native WC bonds were sufficiently separated. \noteb{No energy minimization (no 3spn2 precedent on this). The lammps default emin techniques may not work for this type of CG forcefield. Given the sampling it seems that  1 $\mu$s of equilibration should be sufficient?} Initial bead velocities were assigned from a Maxwell-Boltzmann distribution at the system temperature. \addref{Maxwell-Boltzmann -- \noteb{not sure what to use here, usually not cited?}} Each simulation was conducted for 26 $\mu$s and frame saved to disc every 100 ps, requiring $\sim$24 CPU-hours on 28$\times$Intel E5-2680v4 CPU cores. The first 1 $\mu$s of each run was discarded for equilibration providing us with 40$\times$25 $\mu$s = 1 ms of simulation data for each sequence, during which time we observed 55-100 hybridization/dehybridization events.


\subsubsection{Markov state model construction}

Markov state models (MSMs) are a powerful approach to infer long-time kinetic models from short molecular simulation trajectories \citep{Sengupta2019AutomatedSelf-assembly} \citep{ Pande2010EverythingAsk, Chodera2014MarkovDynamics, Husic2018MarkovScience, Wehmeyer2019IntroductionV1.0} that we employ to construct high-resolution sequence-dependent kinetic models of DNA hybridization and dissociation. MSMs have recently been implemented to study the hybridization mechanism of one particular 14-mer DNA oligomer, but determining the sequence-dependent kinetics and mechanisms was not the focus of this study.\citep{Jin2019} An energy disconnectivity graph-based approach was used to interrogate the differences in hybridization rates and mechanisms between GGGGGG and GCGCGC hexamers to reveal strong deviations from ``all-or-nothing'' behaviors and the importance of zippering and slithering mechanisms.~\citep{Xiao2019} Kinetic models were constructed not from the dynamical trajectories of the molecular model, but by estimating rate constants between local minima were estimated using a transition state theory approximation. Finally, a recent application of MSMs to the long-time dynamics of short RNA oligonucleotides revealed stacking time scales to be highly sequence dependent.~\citep{Pinamonti2017} In this work, MSMs were constructed for each of the four DNA sequences from the 40$\times$25 $\mu$s simulation trajectories following the six-step protocol detailed in Ref.~\citep{Sidky}: (i) trajectory featurization, (ii) dimensionality reduction, (iii) microstate clustering and microstate transition matrix inference, (iv) macrostate clustering and macrostate transition matrix inference. Calculations were performed using the PyEMMA software package. \citep{Scherer2015PyEMMAModels}

\textbf{Featurization.} Trajectories comprising the Cartesian coordinates of the DNA strands as a function of time were featurized using the MDtraj Python libraries \citep{McGibbon2015MDTraj:Trajectories} to represent the system in a manner that exposes the essential system dynamics but eliminates trivial translation and rotational invariances. We adopt pairwise distances $d(i,j)$ between the centers of mass of the 10 bases as a natural rototranslationally invariant featurization that represents each system configuration as the $(10 \times 10)$ 100-element vector of pairwise distances between all distinguishable bases within the complementary sequence pair. This featurization naturally accounts for intermolecular configurations via the pairwise distances between bases on different strands. \noteb{previous text had suggested we used the 190-dim features, but we actually used $100-45 = 55$-dim features since the intramolecular features contributed no additional kinetic variance (I've updated accordingly)} One additional symmetry arises from the self-complementary nature of these sequences -- the sense and anti-sense strands in each pair are identical -- such that the representation of the system under our featurization should remain unchanged upon inverting the arbitrary labeling of strand ``1'' and strand ``2''.\citep{Sengupta2019AutomatedSelf-assembly} The 100-element pairwise distance vector is not invariant to this permutation, but can easily be made so via a simple symmetrization operation in which each of the $10 \choose 2$ =  45 intermolecular pairwise distances are replaced by the mean of the two permutationally invariant distances. Specifically, $\left( d(i_1,j_2) = d(i_2,j_1) \right) \leftarrow 0.5\left( d(i_1,j_2) + d(i_2,j_1) \right)$, where $i_1$ denotes the $i^\mathrm{th}$ base on strand 1 and $j_2$ the $j^\mathrm{th}$ base on strand 2. \citep{Sengupta2019AutomatedSelf-assembly} Finally, we elected to work with the reciprocal of the permutationally-symmetrized pairwise distances to provide higher resolution and differentiation between proximate strand configurations in the near hybridized state compared to distantly separated dissociated strands. VAMP-2 scoring -- calculation of the sum of the squared estimated eigenvalues of the transfer operator -- of trajectories under a particular featurization provides a measure of the kinetic variance carried by that featurization. \citep{Noe2013ASystems, Noe2015KineticSimulation, Scherer2019VariationalKinetics, Wu2020VariationalData} Performing VAMP-2 scoring at a lag time of $\tau$ = 1.2 ns and retaining the top 5 modes, reveals that the reciprocal permutationally-symmetrized pairwise distances can carry up to twice the kinetic variance as the non-reciprocal distances, suggesting that the higher resolution offered at close intermolecular distances can indeed boost the dynamical representational power of the model.

\textbf{Dimensionality reduction.} The featurized trajectories were then projected into a low-dimensional space in preparation for microstate clustering. The standard approach to doing so is to employ time-lagged independent components analysis (tICA) to learn a linear projection into a low-dimensional embedding that maximally preserves the kinetic variance in the data.~\citep{Perez-Hernandez2013IdentificationConstruction, Schwantes2013ImprovementsNTL9, Noe2015KineticSimulation} In this work, we instead employ state-free reversible VAMPnets (SRVs) that can be conceived of as a nonlinear version of tICA. \citep{Chen} SRVs employ neural networks to learn flexible nonlinear functions of the trajectory featurization that better approximate the slow dynamical modes of the system and have been shown to produce substantially higher resolution MSMs than those developed using tICA in applications to WW-domain and Trp-cage mini-proteins.\citep{Chen, Sidky} Importantly, the SRV modes were learned independently for each system to best approximate the slow collective modes for that particular DNA sequence. SRVs were trained using the SRV package we previously developed (\url{https://github.com/hsidky/srv}) employing the default network architecture of two hidden layers each comprising 100 neurons and tanh activation functions, a learning rate of 0.001, and a batch size of 50,000. We adopted a lag time of $\tau$ = 1.2 ns as appropriate short to resolve the dynamical details of the hybridization/dehybridization dynamics. \citep{Phys2018MarkovValidation} As observed by Husic and Pande, it is conceptually incorrect to treat the lag time as a model hyperparameter to be self-consistently optimized via the VAMP-2 score, but rather as a physically-motivated choice designed to expose the dynamical motions relevant at a particular time and length scale of interest. \citep{Husic2017Note:Selection} As we shall show, this choice of lag time leads to converged and Markovian macrostate MSMs. We guarded against overfitting using 5-fold cross-validation in which we divided the trajectory data into 50:50 training:validation splits. We observed plateau of the validation loss and no evidence of overfitting after 20 epochs of training for each system that required approximately 22 GPU-minutes on a single NVIDIA GK210GL GPU card. A VAMP-2 scoring of the cumulative kinetic variance explained as a function of number of SRV collective modes retained also showed no evidence of overfitting -- as would be evinced by separation of the training and validation VAMP-2 scores \citep{Sidky} -- and exhibited a knee for each of the four DNA sequences after the fifth, fourth, third, and second slow modes for AT-all, GC-end, GC-core, and GC-mix, respectively \noteb{not sure the best way to explain this succinctly}. (Figure 1 in the Supporting Information). This motivated the construction of a n-dimensional embedding corresponding to n leading SRV modes retained for each sequence.

\textbf{Microstate clustering.} The SRV projections of the 10 million frames recorded over the course of the 1 ms MD trajectories collected for each DNA sequence were then clustered into microstates using k-means clustering. The VAMP-2 score of the microstate transition matrix constructed for each sequence at the selected $\tau$ = 1.2 ns was highly insensitive to the choice of the number of microstates over the range 100-1000, motivating our selection of 200 microstate clusters for each system.

\textbf{Macrostate clustering.} The 200 microstates comprising each system were finally coarsened into our terminal macrostate MSM. The microstate transition matrix for each system was computed at a range of lag times $\tau$ and then diagonalized to recover the corresponding eigenvalues $\lambda_i$ and associated implied time scales $t_i = -\tau / \ln \left| \lambda i \right|$.~\citep{Wehmeyer2019IntroductionV1.0} The implied time scale plots for the four DNA sequences are presented in SI Figure 2. We observe convergence of the implied time scales very rapidly with lag time $\tau$ for all systems, motivating the construction of high resolution macrostate MSMs at a lag time $\tau$ = 1.2 ns. At this choice of lag time, we recover 5, 4, 2, and 1 implied time scales for the AT-all, GC-end, GC-core, and GC-mid systems, respectively. The identification of $(i-1)$ implied time scales \noteb{(may be 1-2 additional converging, but these correspond to diffusive processes that are irrelevant or unhelpful in determining the hybridization mechanisms. Still, this might come up when looking at the the SI Fig 2} implies the presence of $(i-1)$ slow modes and motivates the coarsening of the system into $i$ macrostates. We estimate these $i$ macrostates by applying PCCA+ spectral clustering to the leading $(i-1)$ eigenvectors of the microstate transition matrix.~\citep{Roblitz2013FuzzyClassification, Weber2018ImplicationsSimulation, Kube2007AConformations} We then estimate the corresponding 6, 5, 3, and 2 macrostate transition matrices $\mathbf{P}$ for the AT-all, GC-end, GC-core, and GC-mid systems, respectively, by projecting the MD simulation trajectories into these discrete macrostates. These macrostate MSMs constitute our terminal kinetic models. We validate the Markovian nature of the four MSMs by subjecting them to the Chapman-Kolmogorov (CK) test \citep{Noe2009ConstructingSimulations,Phys2011MarkovValidation,  Wehmeyer2019IntroductionV1.0}. This test asserts that the transition matrix for a Markovian (i.e., memoryless) MSM constructed at a lag time $\tau$ should satisfy the condition $\mathbf{P}(k \tau) = \mathbf{P}^k(\tau)$, which states that $k$ successive applications of the transition matrix constructed at a lag time $\tau$should be equivalent to a single application of the transition matrix constructed at a lag time $k \tau$. We present in SI Figure 3 the CK tests for each DNA sequence to demonstrate that the $\tau$ = 1.2 ns models performs excellently in predicting transition probabilities out to $k \tau$ = 7.2 ns, validating the Markovian nature and kinetic validity of the four kinetic models.


%\subsection{Featurization}

%A featurization of trajectories was required to capture fundamental dynamics while accounting for translational and rotational symmetries. Pairwise distances between base centers of mass -- similar to heavy atom distances in protein models \citep{Sidky}-- satisfied these requirements and were calculated at each frame using the MDtraj Python library.\citep{McGibbon2015MDTraj:Trajectories} Although pairwise distances remove translational and rotational symmetries, additional symmetries arise from the self-complementary nature of these sequences -- the sense and anti-sense strands in each pair are identical to each other. To account for this, we averaged permutable distances (45 pairs in total) together into a ``symmetrized'' feature set following a similar procedure used in tICAgg coordinate construction.\citep{Sengupta2019AutomatedSelf-assembly} The VAMP-2 scoring method was employed to evaluate the kinetic variance of the feature set and optimize hyperparameters.\cite{ Mardt2018VAMPnetsKinetics} Additional details on the VAMP-2 score can be found in the supplemental information. By comparing scores across several candidate feature sets (Figure \ref{fig:allseq_features_vamp2}), we found that minimal kinetic information was lost in the  symmetrization procedure. Furthermore, the smaller feature set enabled faster training times and better statistics over permutable distances, without appearing to suffer a loss in generality or model resolution.  

%\begin{figure}[ht!]
%	\begin{center}
%        \includegraphics[width=0.55\textwidth]{Fig1.pdf}
%        \caption{A comparison of VAMP-2 scores via 5-fold cross-validation for the intermolecular, intra+intermolecular,  and symmetrized intermolecular distances, in addition to the reciprocal value for each of these coordinates. Reciprocal coordinates performed better across the board, but there was no substantial difference between the three featurization methods. This indicated that minimal kinetic information was lost in the symmetrization procedure, therefore we chose reciprocal symmetrized coordinates to train the model.}
%        \label{fig:allseq_features_vamp2}
%	\end{center}
%\end{figure}

%\subsection{\label{sec:methods}SRVs}
 
%The next step in the MSM pipeline is to create a low-dimensional kinetic projection where state clustering is performed. While most pipelines employ tICA to generate a linear transformation of input features, we use SRVs as a more powerful modular replacement for tICA. SRVs were first developed by Chen et al. as a means to directly learn slow eigenfunctions of the transfer operator.\citep{Chen} The method is a descendant of VAMPnets\citep{Mardt2018VAMPnetsKinetics}, deep canonical correlation analysis \citep{Andrew2013DeepAnalysis}, and extended dynamic mode decomposition with dictionary learning (EDMD-DL).\citep{Li2017ExtendedOperator} The framework uses a twin-lobed artificial neural network to learn an optimal basis set for the variational approach to conformational dynamics (VAC) from which the leading eigenfunctions of the transfer operator are then estimated.\citep{Noe2013ASystems} The resulting orthogonal modes are associated with the slowest dynamical processes in a system, and can be used to interpret kinetic information directly (such as physical correlations and time scales) and to construct MSMs.\citep{Sidky, Pande2010EverythingAsk} SRVs provide robust nonlinear approximation and computation time that scales linearly with the amount of input data and are more powerful and efficient that tICA and ktICA approaches. This is a key attribute to our system as 10 million frames with 55 features per frame are used for each sequence. The SRV framework has been used to construct high kinetic resolution MSMs applied to molecular simulations of WW-domain and Trp-cage mini-protein.\citep{Chen, Sidky}

%\subsection{\label{sec:methods}SRV-MSMs}

% need some more detail here -- talk to Andy about this
%MSMs are a powerful tool for interpreting large amounts of simulation data in a statistically robust and experimentally comparable way.\citep{Phys2011MarkovValidation, Husic2018MarkovScience} Kinetically similar conformations are first discretized into microstates, and the conditional probability of transitions between microstates is calculated within some lag time. The reliance on conditional probabilities allows for many independent simulations (longer than the lag time) to be collectively interpreted. To take full advantage of the MSM frameworks, however, the input basis should be as kinetically meaningful as possible.\citep{Pande2010EverythingAsk} Because SRV eigenfunctions translate simulation features into their slowest kinetic representations, they function as an optimal MSM basis. To build our SRV-MSM framework, we employed the PyEmma MSM pipeline and generated independent models for each sequence.\citep{Scherer2015PyEMMAModels} In a similar approach to Sidky et al. we performed k-means microstate clustering, Bayesian MSM construction, and PCCA+ hierarchical macrostate assignments.\citep{Sidky} The number of microstates were determined by VAMP-2 score, and the SRV-MSM lag time was selected based on implied time scales convergence.  The number of PCCA+ macrostates was determined based on the characteristic of each system and is discussed more in depth in the results and supplemental information.

%\subsection{\label{sec:methods}Diffusion Maps}
%Diffusion maps generate a low-dimensional non-linear embedding of data into high variance structural modes.\citep{Coifman2006DiffusionMaps, Ferguson2010SystematicMaps} They are well-suited to find subtle differences in molecular conformations that are temporally disconnected or that interchange faster than the MSM lag time. We used reciprocal inter-molecular base distances as inputs to diffusion maps, and tuned $\epsilon$ and $\alpha$ parameters in accordance with previous studies.\citep{Shmilovich, Wang2017MesoscaleLandscapes} We found that the choice of these parameters mainly served to condense or stretch the low dimensional space and had a minimal qualitative impact on salient results.

%%%%%%%% Methods comments %%%%%%%%%%%%%%%%%

%We found close agreement with explicit ion simulations performed with 240 mM \ce{NaCl} and 18 mM \ce{MgCl2} (supplemental) but 7x slower simulation speed made adequate sampling more difficult to achieve under these conditions.

%We initialized explicit ions such that 240 mM \ce{NaCl} and 18 mM \ce{MgCl2} were added to the box in addition to 18 Na counter ions to balance the charge from the 9 phosphate groups in each oligonucleotide backbone \citep{Hinckley2015}. We used a periodic box size of 7.774 nm and an effective oligo concentration of 7 mM. We used an Ewald summation to calculate long range Coulombic interaction between DNA and ions using a real space cutoff of 2 nm.

\subsection{Experimental Methods}

\subsubsection{Sample Preparation}

Each DNA oligonucleotide was purchased from Integrated DNA Technologies (IDT) at desalt grade purity. Oligonucleotides were purified with 3 kD cutoff centrifugal filters (Amicon). All labile protons were exchanged in deuterium oxide (\ce{D2O}, Cambridge Isotopes, 99.9\%). Oligonucleotides were prepared at a total strand concentration of 2 mM in 50 mM pD 7.2 sodium phosphate buffer with 240 mM NaCl and 18 mM \ce{MgCl2}. Prior to each measurement, DNA solutions were placed in a water bath at 90$^{\circ}$ C for 3 minutes and then cooled to room temperature under ambient conditions.

\subsubsection{T-Jump IR Spectroscopy}\label{sec:Tjump}

The details of the technique and processing used to acquire temperature-jump infrared (T-jump IR) data have been described previously.~\citep{Chung2007TransientKinetics,Jones2009Heterodyne-detectedSpectroscopy,Jones2012TransientInfrared} Briefly, heating was initiated through optical excitation of the O-D stretch overtone transition of \ce{D2O}. The 1.98 $\mu$m pulses (5 ns, 20 mJ, 20 Hz) used for heating were generated from the frequency-doubled output of a Nd:YAG (YG 980, Quantel) later sent through an optical parametric oscillator (OPO). Nonlinear IR spectra are collected from 5 ns to 50 ms delays after the T-jump. T-jump heterodyne-detected vibrational echo (t-HDVE) IR spectra were acquired with Fourier transform spectral interferometry,~\citep{Jones2009Heterodyne-detectedSpectroscopy} where the delay between the local oscillator (LO) and DVE signal was scanned in 5 fs steps between -10 and 10 fs. t-HDVE spectra were acquired with a parallel pulse polarization scheme. Data are presented using the recovered T-jump dispersed pump-probe (t-DPP) spectrum. t-DPP data are reported as the difference spectra with the 49 or 50 ms spectrum, which corresponds the initial temperature spectrum. 

The sample was placed between two 1 mm thick CaF2 windows separated by a 50 $\mu$m Teflon spacer enclosed in a home-built brass jacket. The initial temperature of the sample was set using a recirculating chiller connected to the brass sample jacket. The T-jump temperature change ($\Delta$T) was set to $\sim$15 $^\circ$C for all measurements and monitored using the change in transmission of the \ce{D2O} bend-libration combination band measured in the LO beam. The temperature change was quantified by comparing the change in transmission of the LO beam with a FTIR temperature series of \ce{D2O}. 

\subsubsection{Determination of Fast and Slow Dissociation Rates}\label{sec:TjumpAnalysis}

To determine observed rates from the T-jump data, the time-domain t-HDVE data was inverse Laplace transformed into the rate domain using a maximum entropy approach (MEM-iLT).~\citep{Kumar2001OnTimescales} Observed rates were taken as the amplitude-weighted mean rate across detected IR frequency, as previously described.~\citep{Sanstead2018DirectDehybridization} The fast response ( $k_d^\mathrm{fast}$) is defined purely as this amplitude-weighted mean rate, whereas the dissociation rate constant ($k_d^\mathrm{slow}$) was extracted from the observed rate of the slow response ($\lambda^\mathrm{slow}$) using a two-state model for self-complementary oligomers.~\citep{Bernasconi2012RelaxationKinetics}
	\begin{equation}\label{e1}
	\lambda^\mathrm{slow} = k_d^\mathrm{slow} + 4[S]_{T_f}k_a
	\end{equation}
$[S]_{T_f}$ is the concentration of single-strand oligomer at the final temperature of the T-jump, and $k_a$ is the association rate constant. In practice, Equation~\ref{e1} is recast in terms of the dissociation equilibrium constant ($K_d$) to solve for $k_d^\mathrm{slow}$ and $k_a$.
	\begin{equation}\label{e2a}
	k_d^\mathrm{slow} = \frac{\lambda^\mathrm{slow}K_d}{K_d+4[S]_{T_f}}
	\end{equation}
	\begin{equation}\label{e2b}
	k_a = \frac{k_d}{K_d}
	\end{equation}
$k_d^\mathrm{slow}$ and $[S]_{T_f}$ come from melting curves measured with FTIR spectroscopy that are fit to a two-state all-or-nothing dissociation model.~\citep{Marky1987CalculatingCurves} We used the FTIR melting curves previously reported in Ref.~\citep{Sanstead2016}





%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%


\section{\label{sec:Results}Results}

\subsection{Sequence-dependent coarse-grained kinetics recapitulate T-jump IR measurements} \label{sec:time}

% rerun these at the actual final temps and just see what happens? Doesn't really matter where Tjump occurs and could make things easier as a whole...
% talk about rerunning AT-all and GC-mix=

We first sought to demonstrate that the coarse-grained 3SPN.2 model accurately recapitulates the experimentally observed kinetics of DNA oligomer hybridization/dissociation by validating our computational predictions against temperature-jump infrared (T-jump IR) experiments. We conducted T-jump IR experiments for each DNA sequence as a function of temperature and extracted the ``slow'' (10-30 $\mu$s) and ``fast'' (70-100 ns) observed rates in the T-jump response (Section \ref{sec:Tjump}). As discussed in Section \ref{sec:TjumpAnalysis}, the fast ($k_d^\mathrm{fast}$) experimental rate is an observed rate whereas we have derived a dissociation rate constant ($k_d^\mathrm{slow}$) for the slow process.\noteb{is this stipulation clear enough?} The slow response has previously been attributed to duplex dissociation on microsecond time scales induced by the rapid heating of the initially hybridized duplex.~\citep{Sanstead2016, Sanstead2018DirectDehybridization} The fast response is has been assigned to terminal base pair fraying,~\citep{Sanstead2016, Sanstead2018DirectDehybridization} with this process corresponding to a relatively complex dynamical process that can span time scales from picoseconds to microseconds,~\cite{Nonin1995TerminalFraying, Nikolova2012ProbingSimulations, Andreatta2006UltrafastHelix, Galindo-Murillo2015ConvergenceDGCACGAACGAACGAACGC} and all-atom simulations suggest that frayed ends can assume misaligned Watson-Crick bonds, base-sugar hydrogen bonds, and terminal stacked conformations.~\citep{PinamontiTheModels, Zgarbova2014BaseRNA} 

To computationally mimic the T-jump process in our 3SPN.2 simulations, we conducted 1 $\mu$s simulations of an initially hybridized DNA duplex over a range of temperatures and monitored its structural relaxation. We performed 120 independent simulations for each DNA sequence at each temperature, and from these extracted computational estimates of $k_d^\mathrm{slow}$ and fast $k_d^\mathrm{fast}$ (SI Figure 4). First, we tracked the slow response corresponding to duplex dissociation in our simulations by compiling the distribution of times at which the central base pairs first separate to a distance of 1.3 nm. This cutoff was selected as the distance beyond with the strands are effectively non-interacting and defines the dissociated state. We extracted our computational estimate of $k_d^\mathrm{slow}$ by fitting a decaying exponential to the fraction of hybridized sequences as a function of time $f_\mathrm{hybridized}(t) = \exp(-k_d^\mathrm{slow}t)$. Second, we tracked the fast response corresponding to terminal base pair fraying by compiling the distribution of times at which either of the terminal base pairs first separated to a distance of 1.3 nm, corresponding to a complete breakage of the Watson-Crick (WC) interaction. We extracted our computational estimate of $k_d^\mathrm{fast}$ through a decaying exponential fit to the fraction of unfrayed sequences as a function of time $f_\mathrm{unfrayed}(t) = \exp(-k_d^\mathrm{fast}t)$. 

We present in Fig.~\ref{fig:relaxation-comparison} a comparison of $k_d^\mathrm{slow}$ and $k_d^\mathrm{fast}$ estimated by computation and experiment. Although the 3SPN.2 model was not directly fitted against kinetic data,~\cite{Hinckley2013AnHybridization} its predictions of sequence dependent T-jump relaxation rates are, within a systematic scaling factor in time and systematic shift in temperature, in good agreement with observed experimental trends. It is well-known that the smoothing of the underlying free energy landscape induced by coarse-graining artificially accelerates the kinetics of coarse-grained molecular simulations and that different degrees of freedom may be accelerated by different factors.~\citep{Marrink2007TheSimulations, Fritz2011MultiscaleDynamics, Marrink2013PerspectiveModel} We find that the simulated slow responses corresponding to center-of-mass translation of the strands during dissociation of the duplex is $\sim$10$\times$ accelerated relative to experiment, whereas the fast responses corresponding to fraying of the terminal bases is $\sim$120$\times$ accelerated. We apply these sequence-independent scaling factors to our reported computational values. Although the 3SPN.2 model reproduces melting temperatures relatively well, we observed a systematic 4 K under-prediction in making the experimental comparison and so we apply a universal +4 K shift corrective calibration to our computational results. We note that these empirical calibration factors to the 3SPN.2 predictions are applied only for the purposes of making an experimental comparison. The computational time scales and melting temperatures reported in the remainder of the paper are not corrected since we are only interested in the relative trends in the behaviors of the four sequences.

\begin{figure}[ht!]
	\begin{center}
        \includegraphics[width=\textwidth]{Fig1.pdf}
        \caption{Experimental measurements and computational predictions of slow and fast at T-jump IR responses. Experimental results are shown in terms of the final T-jump temperature (a) The simulated and T-jump dissociation rate constants $k_d^\mathrm{slow}$ corresponding to duplex dissociation over long time scales. (b) The Simulated fraying rate constant and fast observed rate $k_d^\mathrm{fast}$ \noteb{again trying to make the distinction between observed rate and dissociation rate here} corresponding to terminal base-pair fraying on short time scales. The simulation results are corrected by a sequence-independent scaling factor that corrects for a 10$\times$ acceleration of the slow dissociation dynamics and 120$\times$ acceleration of the fast fraying dynamics. The simulated temperature in all cases is also corrected by +4 K to correct for an observed systematic under-prediction of the melting temperature by the 3SPN.2 model}
        \label{fig:relaxation-comparison}
	\end{center}
\end{figure}


In Fig.~\ref{fig:relaxation-comparison}a we report an exponential increase of $k_d^\mathrm{slow}$ with temperature, as expected from the large enthalpic barrier to duplex dissociation.\cite{Craig1971RelaxationOligonucleotides,Porschke1971CooperativeTransition,Williams1989LaserDGCATGC} Under the time and temperature calibration corrections, we see generally very good agreement between the computational and experimental curves. The computational rates do tend to systematically under-predict the experimental values at low temperatures, an effect that we ascribe to the fact that the experimental T-jump responses likely contain a mixture of dissociation and hybridization events -- the latter of which persist to lower temperatures than the former -- whereas the computational analysis considers dissociation alone. Of the four sequences, GC-core shows the largest discrepancy between computation and experiment, although the general trends exponential trend is preserved. This may be a result of its high propensity to fray as we will discuss further in Section \ref{sec:GC-core} \noteb{Is this sufficient discussion here since we bring it back up later?} In Fig.~\ref{fig:relaxation-comparison}b we expose a largely linear dependence of $k_d^\mathrm{fast}$ upon temperature for AT-all, GC-end, and GC-mix compared to an exponential dependence for GC-end. This can be understood via the comparatively larger enthalpic barrier for dissociation of the terminal G:C base pair in GC-end compared to that for the A:T terminal pair in the other three sequences. Again we see good agreement between the scaled computational predictions and the experimental T-jump measurements. The favorable comparison of computation and experiment provides support for the capacity of the 3SPN.2 model to reliably reproduce sequence-dependent trends in the slow and fast kinetics of the four DNA oligomers.

%Coarse-grained time scales are inherently difficult to match against experiments due to smoothing of the free energy landscape and variable acceleration across different degrees of freedom. As such, we evaluated multiple temperatures for each sequence and inferred acceleration factors from temperature-dependent trends.  Experimental temperature-dependent relaxation time scales were measured for each sequence using T-jump IR spectroscopy as described previously.\citep{Sanstead2018DirectDehybridization} We recorded ``fast'' and ``slow'' amplitude-weighted responses -- previously attributed to terminal base pair fraying and duplex dissociation, respectively -- for each temperature and sequence. To compare results with 3SPN.2, we ran 120 shorter (1 us) simulations initialized in the hybridized state at final T-jump temperatures for each sequence. We calculated relaxation times for a slow dissociation response and fast fraying response at each temperature and compared these with experimental temperature-dependent relaxation fits.

% need to discuss tmelt shift
%We measured the slow dissociation response by fitting the distribution of times at which the core base pairs separate beyond a 1.3 nm cutoff. We found the inverse of these relaxation times -- the effective dissociation rate -- to increase exponentially with temperature, which is expected given the large enthalpic barrier of dissociation.\citep{Craig1971RelaxationOligonucleotides,Porschke1971CooperativeTransition,Williams1989LaserDGCATGC} We noted an average temperature shift of about 4$^{\circ}$C between experiment and simulation fits. While 3SPN.2 captures melting temperature relatively well, some deviation is expected given varied ion conditions and other coarse-grained effects. We also noted an acceleration of about one order of magnitude compared to experiment, although this factor is sensitive to the exact definition of melting temperature. After accounting for the melting temperature shift and acceleration factor, we saw strong agreement between the experimental data and simulated relaxation fits (Figure \ref{fig:relaxation-comparison})). At lower temperatures, T-jump slow responses likely contain a mixture of dissociation and hybridization dynamics, therefore rates do not drop off to the same extent as our analysis, which considers dissociation alone. GC-core showed the largest deviation from experiment, indicating that increased temperature triggers dissociation at a disproportionately higher rate in simulation compared to experiment. 

% GC-core fast time scales could be slower as they have contributions from internal fraying which would have a longer relaxation than the termini alone.
% in order to make comparisons across these different free energy landscapes, we adjusted simulations by a consistent acceleration factor. We maintained a consistent accegqrelation factor across sequences, but found that optical scaling different by about one order of magnitude between fast and slow responses. It is expected given that different degrees of freedom often experience differing amounts of accelerations in coarse-grained models.

% need to update this fig with labels and temp shift

% A documented short-coming for the 3SPN.2 model is a high proclivity for termini fraying relative to what has been observed in vitro \citep{2013}. This supports why we observe significantly more acceleration for the fast response, but relative sequence and temperature effects still compare well with T-jump results.

%The fast response -- which Sanstead et. al attributed to terminal base pair fraying signatures -- was more challenging to compare against simulation observables. Numerous experimental and computational studies have shown that DNA and RNA fraying is a complex dynamical process with time scales that span 5 ps to several microseconds.\citep{ Nonin1995TerminalFraying, Nikolova2012ProbingSimulations, Andreatta2006UltrafastHelix, Galindo-Murillo2015ConvergenceDGCACGAACGAACGAACGC} All-atom simulations suggest that frayed ends can assume misaligned WC bonds, base-sugar hydrogen bonds, and terminal stacked conformations.\citep{PinamontiTheModels, Zgarbova2014BaseRNA} Given that there is only one interaction site parameterized on each 3SPN.2 base, we would not expect to resolve this diverse collection of states and dynamics. Instead, we measured the fast fraying response by counting frames until duplex terminal ends splits beyond a cutoff. This approach assumes that fraying on the permutable top and bottom of the duplex are independent from each other, and that a base pair distance is a reliable approximation for the ensemble spectroscopy signal. This is a reasonable assumption given that the amplitude-weighted time scales should consist largely of terminal fraying events. Again we fit relaxation curves to the ensemble of fraying time scales in order to extract rate approximations for each sequence at a series of temperatures. 

%For A:T terminal sequences, the simulated fast response appears linear with temperature, indicating a barrierless and diffusion-driven process. In contrast, the GC-end responses are distinctly slower and increase exponentially with temperature, likely due to a greater enthalpic barrier associated with G:C fraying. We observe similar trends in the experimental data, although comparisons at the highest temperatures were limited due to mixing between the fast and slow responses (Figure \ref{fig:relaxation-comparison}). We found the optimal acceleration factor to be dependent on the choice of the cutoff parameter, but we can approximate that 3SPN.2 fraying dynamics are accelerated by about 100-200x relative to experiment. It is not surprising that we see different rates of acceleration for the dissociation and fraying processes given that coarse-grained effects can vary across different degrees of freedom. In particular, the simplified treatment of the fraying process may smoothen the free energy landscape and speed up dynamics relative to a more global processes like dissociation. Although one should rely on higher resolution models to study in depth mechanisms of fraying, our results indicate that terminal fraying is a reasonable assignment for the spectroscopic fast response in Sanstead et al. and that 3SPN.2 can recapitulate some sequence-dependent fraying effects.
% need a better transition here back into SRV-MSMs?



\subsection{SRV-MSMs for each sequence}

We now proceeded to construct Markov state models (MSMs) from 1 ms of aggregated simulation trajectories for each of the four sequences at their respective melting temperatures to generate sequence-dependent long-time kinetic models. MSMs are extremely valuable in providing both a quantitative predictive model and a physically comprehensible mechanistic understanding of the long-time dynamical evolution of the system between an ensemble of metastable macrostates. We present in Fig.~\ref{fig:allseq_table} the inferred MSMs for each of the four 10-base DNA sequences. Across all four sequences we identify a totality of seven metastable macrostates corresponding to the fully hybridized state (H) in which all native base pairings are intact, four shifted states in which the strands are translated out-of-register by two or four bases in the 5$^\prime$ (5S2, 5S4) or 3$^\prime$ (3S2, 3S4) direction, a frayed state (F4) -- unique to GC-core -- in which the four terminal A:T base pairs are unbound, and the fully dissociated state (D). We present these seven macrostates in Fig.~\ref{fig:allseq_table}a along with schematic and cartoon renderings of representative microstates contained within each of these macrostates. In Fig.~\ref{fig:allseq_table}b we present the occupancy probabilities of each state at thermodynamic equilibrium. By virtue of the fact that each sequence is simulated at its corresponding melting temperature (T$_\mathrm{m}$), the probability of the dissociated state (D) is, by construction, approximately equal to the sum of the probabilities over the remaining six states (H, 5S2, 3S2, 5S4, 3S4, F). In Fig.~\ref{fig:allseq_table}c we present a visualization of the macrostate MSMs for each of the four sequences showing the connectivity between the identified macrostates. The macrostates are represented as orange circles in proportion to their equilibrium probabilities and the grey arrows indicate the probability of hopping from one macrostate to another under one time step of the kinetic model. The identity and fluxes between the macrostates provide a wealth of high-level, interpretable information on the sequence-dependent metastable states and hydbrization/dehybridization pathways. Immediately we identify that the AT-all sequence possesses a rich and complex dynamical landscape comprising six metastable states whereas GC-mix exhibits far simpler two-state ``all-or-nothing'' behavior. In Fig.~\ref{fig:allseq_table}d we present the so-called implied time scales of each MSM. These time scales correspond to the relaxation times of the DNA dimer among its constituent metastable macrostates. The leading implied time scale for each system corresponds to the characteristic time scale for hybridization/dehybridization. Since each system is simulated at its respective melting temperature it is not surprising that the leading time scale is approximately equal for all four systems. The spectrum of higher order time scales corresponds to increasingly quicker relaxations between the metastable macrostates. The total number of implied time scales is typically one fewer than the number of metastable macrostates and the existence of large implied time scales is indicative of slowly relaxing kinetic processes. The dense spectrum of slow implied time scales for AT-all is indicative of its relatively complex kinetic landscape whereas that for the two-state GC-mix comprises only a single time scale corresponding to hybridization/dehybridization. We now proceed to analyze in detail the sequence-dependent thermodynamics, kinetic, and mechanisms exposed by the four MSMs.

%We followed the SRV-MSM analysis pipeline described in the Methods section to generate sequence-dependent kinetic models for each oligomer. We identified seven kinetically relevant states that were captured within the resolution of the model. Figure \ref{fig:allseq_table} shows cartoon and molecular renderings for each of these states as well as their sequence-specific stationary probabilities.  Figure \ref{fig:allseq_transition} shows how these states fit into sequence-specific low-dimensional representations and kinetic models. Arrows between states show the probability of a transition to another state within the MSM lag time. 

%Shifted states are only observed for AT-all and GC-end; state abbreviations indicate the direction of the shifted overhang (5$^\prime$ vs. 3$^\prime$) and the number of shifted motifs relative the native state (1 vs. 2). GC-mix displays two-state behavior within the resolution of the model, however we will show that the hybridization and dissociation are still characterized by an ensemble of shorter-lived states. Stationary probabilities in each of these states are shown in figure \ref{fig:allseq_table}. We also show that PCCA+ clustering assigns states to relative free energy minima in the tICA space.

% rewrite this into one paragraph

% change state names and scientific notation
% include D here? can see what it would look like...

\begin{figure}[ht!]
	\begin{center}
        \includegraphics[width=170mm, scale=1]{Fig2.pdf}
	\end{center}
\end{figure}
\begin{figure}[ht!]
        \caption{ Summary of thermodynamics and kinetics quantities extracted from sequence-specific MSMs. (a) Seven metastable macrostates sampled by at least one of the four sequences. Cartoons represent 10bp sequences in the 5$^\prime$ to 3$^\prime$ direction, where red-to-blue contacts indicate WC base pairing and black indicates an unbound bases. Molecular rendering show 3D examples of each configuration. (b) Number of frames (out of $10^7$ total) clustered into each of the seven macrostates. Uncertainties are calculated across 100 MSMs using a Bayesian MSM estimator and are very small compared to total populations. (c) Coarse-grained MSMs and molecular renderings for each sequence. Arrow thickness corresponds to the flux between between states (using of minimum flux cutoff of $3\times10^{-6}$), and the overlaid number represents the probability of making a transition within the 1.2 ns lag time. The radii of state circles correspond to the log of state populations. (d) Distribution of leading timescales for each sequence. The leading timescales for each sequence corresponds to hybridization/dehybridization process. AT-all and GC-end display a spectrum of converged slow modes corresponding to shifting processes. GC-core has one faster timescale corresponding to fraying processes.
        \noteb{Previous (more compact) version Fig2 alternate}
    }
    \label{fig:allseq_table}
\end{figure}



\subsection{Comparison of MSM thermodynamic predictions with NN model} \label{sec:NN}

We first compare the thermodynamic predictions for the equilibrium macrostate probabilities to those of predictive sequence-dependent models for the thermodynamics of DNA association. The nearest neighbor (NN) model is popular model of DNA hybridization thermodynamics that predicts the free energy of duplex formation as a sum over helix initiation terms and the hybridization free energies of nearest neighbor pairs of bases that account for both the specific WC pairings and the modulating effects of the local (i.e., nearest neighbor) environment.~\citep{SantaLucia1998AThermodynamics, Santalucia2004TM} The parameters of the NN model were estimated by regressing over 108 experimental measurements to furnish a predictive model for the free energy of association as a function of DNA sequence and explicitly account for stacking contributions of native pairs, internal mismatches, and dangling ends. We apply the NN model to estimate the (relative) free energy $F$ of each of the macrostates occupied by each of the four sequences. The free energy of each macrostate is related to its equilibrium occupancy probability $P$ via the statistical mechanical relationship $F = -k_B T \ln P + C$, where $T$ is temperature, $k_B$ is Boltzmann's constant, and $C$ is an additive constant reflecting our ignorance of the absolute scale of free energies. Since the additive constant cancels out in the evaluation of relative free energies between macrostates, this calculation provides a means to compare the free energies predicted by our MSMs (Fig.~\ref{fig:allseq_table}) to those of the NN model. A full accounting of the application of the NN model to the macrostates occupied by each of the four sequences is provided in the Supporting Information.

As illustrated in Fig.~\ref{fig:NN_table}, we see that the NN model tends to predict lower free energies for each macrostate compared to the MSM and therefore overestimate macrostate stability relative to the hybridized (H) state by as much as 14.7 kJ/mol. (The free energy of the hybridized state is set to zero by construction for both the MSM and NN models.) The only exception to this trend is the frayed state (F4) unique to CG-core, where the MSM and NN predictions are in good agreement. Although the trends are in qualitative agreement, what is the root of the quantitative discrepancy of the MSM and NN models in the 5S2, 3S2, 5S4, and 3S4 states? First, the MSM is constructed bottom-up from molecularly detailed 3SPN.2 simulations whereas the NN model is fitted top-down by regression against experimental data. There are approximations inherent in the 3SPN.2 model and the NN model was fitted to limited experimental data assuming a low-order expansion in terms of nearest neighbor additive contributions. Second, a well known deficiency of the NN model is the absence of any treatment of inert tails -- free bases that extend beyond the dangling end are known to destabilize the duplex.\citep{Michele2014EHybridization} Visualizations show that, although free A:T ends in the F4 state could be considered inert tails, they tend to extend away from the intact section of the duplex and therefore may have less of an impact on duplex stability. \noteb{not sure the best way to phrase this, but they could technically still be considered inert tails}. In this case the NN and MSM free energy predictions are in good agreement. Two of our sequences, AT-all and GC-end, occupy macrostates 5S2 and 3S2 comprising a one-base inert tail and 5S4 and 3S4 comprising a three-base inert tail (Fig.~\ref{fig:allseq_table}). For both sequences, our simulations show that 5S2 is both the most stable of shifted states and has the smallest offset (4.0 kJ/mol for AT-all and 10.0 kJ/mole for GC-end) compared to NN predictions \ref{fig:NN_table}b. The 3S2 and and 5S4 states have nearly the same deviation from NN predictions (7.0-7.2 kJ/mol for AT-all and 14.3-14.7 kJ/mole for GC-end), indicating that longer tails in the 5$^\prime$ direction are as destabilizing as shorter tails in the 3$^\prime$ direction. It is known from differential scanning calorimetry studies~\citep{Dickman2012ThermodynamicDNAs} that 3$^\prime$ inert tails are more destabilizing than 5$^\prime$ tails, with the differential behavior attributed to a combination of 5$^\prime$ tails preferentially stacking on the core duplex and 3$^\prime$ tails perturbing the duplex structure. \citep{Doktycz1990ThermodynamicATGC, Dickman2012ThermodynamicDNAs, Michele2014EHybridization}. Both the NN and MSM predictions for AT-all are consistent with this trend (i.e., $F_\mathrm{3S2} > F_\mathrm{5S2}$ and $F_\mathrm{3S4} > F_\mathrm{5S4}$). For GC-end, the MSM and NN models both predict the four shifted states to be less stable than for AT-all. This is in line with expectations since the terminal G:C pairs in GC-end decrease the number of available WC pairings by two compared to AT-all. Given that terminal mismatches tend to be slightly stabilizing \citep{Santalucia2004TM}, we might expect a lower discrepancy than the  10.1-14.7 kJ/mol offset that we observe between NN and MSM results. This offset may arise in part from the excluded volume interactions used to account for mismatched bases in 3SPN.2, however we attribute it mainly to inert tail effects. The GC-end NN predictions run contrary to the expectation that the 3$^\prime$ inert tails should be more destabilizing than the 5$^\prime$ tails, whereas the MSM predictions are consistent with this trend. Indeed, the MSM model for GC-end does not even register the 3S4 macrostate as a stable metastable conformation for the duplex.

In sum, the qualitative trends in the macrostate thermodynamic stabilities are in good agreement between the MSM and NN models, but show quantitative discrepancies for macrostates possessing inert tails. In these instances the NN model overestimates macrostate stability relative to the hybridized state by 4.0-14.7 kJ/mol. The MSM predictions are also consistent with the experimental expectation that 3$^\prime$ inert tails should be more destabilizing than the 5$^\prime$ tails, whereas the NN predictions can be in conflict with this trend.

\noteb{Wrote up a different approach to this section should we present the NN data with no dangling ends instead. Have not fully pieced it together, but this is the basis idea:}

\bood{As illustrated in Fig.~\ref{fig:NN_table_no_de}, we see that our macrostate free energies recapitulate NN model predictions reasonably well. It is important to note that we do not include dangling end contributions when calculating NN free energies. Although 3SPN.2 can capture some dangling end stabilization effects through base stacking and cross-stacking interactions, the CG model was not parameterized to fully capture the enthalpic contributions of free bases interaction with terminal base pairs. For each macrostate, dangling ends would be expected to have stabilizing effects \noteb{Can show NN with dangling end contributions (original figure) in SI)} and therefore increase state populations compared to what we observe in simulation.}

\bood{A well known deficiency of the NN model is the absence of any treatment of inert tails -- free bases that extend beyond the dangling end are known to destabilize the duplex. We find that inert tail effects may be particularly impact in predicting out-of-register state stability.}

\bood{For the AT-all sequence, we see close agreement between the macrostate and NN prediction for the 5S2 state. However, we observe deviations of 3.5 and 2.3 kJ/mol the 3S2 and 5S4 states, respectively. This is consistent with UV absorbance spectroscopy and differential scanning calorimetry experiments~\citep{Dickman2012ThermodynamicDNAs, Michele2014EHybridization} which show that (i) 3$^\prime$ tails are more destabilizing than 5$^\prime$ tails and (ii) longer tails are more destabilizing than shorter tails. These effects are compounded for the 3S4 state, where we observe an MSM free energy 5.1 kJ/mol higher than the NN model prediction.}

\bood{For GC-end, the MSM and NN models both predict the four shifted states to be less stable than for AT-all. This is in line with expectations since the terminal G:C pairs in GC-end decrease the number of available WC pairings by two compared to AT-all. The deviations between MSM and NN models follow a similar trend based on inert tail effects, but these are also consistently higher -- ranging from 3.2 to 7.8 kJ/mol. Indeed, the MSM model for GC-end does not even register the 3S4 macrostate as a stable metastable conformation for the duplex. We attribute this additional destabilization to excluded volume interactions between mismatched terminal pairs in the model, as complex and slightly stabilizing~\citep{Santalucia2004TM} terminal G:A and C:T interactions are not included in 3SPN.2.}

\bood{Although the NN model tends to under-predict the MSM free energy for AT-all and GC-end, we see that the GC-core F4 state is about 2.5 kJ/mol more stable than predicted by NN. This can be attributed to an added probability of an ensemble of ``double-frayed'' state in which 1-2 terminal A:T base pairs are frayed and the configuration is clustered into F4.}

\noteb{Can add something here (or in kinetic section) about how, given the MSMs seem not to account for dangling end stability, there would likely be a higher population of each state than we observe in experiment. This could (i) increase out-of-register role in the hybridization process my increasing membership in intermediate states or (ii) decrease the likelihood of shifting behavior due to higher barriers between those states.}


\begin{figure}[ht!]
	\begin{center} 
        \includegraphics[width=160mm, scale=1]{Fig3.pdf}
        \caption{Comparison of the macrostate free energy predictions of the MSMs and nearest neighbor (NN) thermodynamic model.~\citep{SantaLucia1998AThermodynamics, Santalucia2004TM} The arbitrary additive constant to the free energy is specified such that the free energy of the hybridized macrostate is zero. With the exception of the frayed state (F4), the NN model predicts lower free energies than the MSM and therefore tends to overestimate macrostate stability relative to the hybridized state. (a) Free energy estimates relative to the hybridized state for both the simulated macrostates and corresponding NN prediction. (b) Free energy difference between the NN prediction and simulated value for each macrostate.}
        \label{fig:NN_table}
	\end{center}
\end{figure}

\begin{figure}[ht!]
	\begin{center} 
        \includegraphics[width=160mm, scale=1]{Fig3_no_DE.pdf}
        \caption{Comparison of the macrostate free energy predictions of the MSMs and nearest neighbor (NN) thermodynamic model.~\citep{SantaLucia1998AThermodynamics, Santalucia2004TM} The arbitrary additive constant to the free energy is specified such that the free energy of the hybridized macrostate is zero. These NN estimates do not include additional thermodynamic stabilization from dangling ends \noteb{can include this in SI?} (a) Free energy estimates relative to the hybridized state for both the simulated macrostates and corresponding NN prediction. (b) Free energy difference between the NN prediction and simulated value for each macrostate. \noteb{Should the y-axis be more descriptive here?}}
        \label{fig:NN_table_no_de}
	\end{center}
\end{figure}


%The thermodynamic stability of these states can be evaluated against experimental prediction by defining each structure in terms of ``dangling ends'' -- unpaired bases adjacent to the paired duplex -- and ``inert tails'' -- free bases that extend beyond the dangling end \citep{Michele2014EHybridization}. Dangling ends tend to have small stabilizing effects, and inert tails decrease stability as they increase in length -- 3$^\prime$ inert tails have stronger effects than 5$^\prime$ tails. Although sequence-dependent effects of inert tails are less well known, dangling end effects can be factored into nearest neighbor (NN) calculations \citep{Santalucia2004TM}. 

% and inert tails tend to decrease stability as they increase in length, particularly at lower ionic strengths. It has been shown that 5$^\prime$ dangling ends with one inert tail have higher melting temperatures and are energetically favorable compared to 3$^\prime$ ends \citep{Senior1988InfluenceDuplexes, Dickman2012ThermodynamicDNAs}. These effects might be attributed to some combination of 5$^\prime$ tails preferentially stacking on the core duplex and 3$^\prime$ tails perturbing the duplex structure \citep{Doktycz1990ThermodynamicATGC}. 

% This leads us to believe that 5$^\prime$ vs. 3$^\prime$ differences -- attributed to some combination of 5$^\prime$ tails preferentially stacking on the core duplex and 3$^\prime$ tails perturbing the duplex structure -- may out-weigh NN effects \citep{Doktycz1990ThermodynamicATGC}. 


\subsection{Out-of-register states facilitate hybridization and dissociation dynamics (AT-all, GC-end)}

In addition to thermodynamic stabilities, the macrostate MSM also furnishes quantitative and interpretable predictions of hybridization and dehybridization pathways and mechanisms. We now proceed to analyze these predictions for each of the four sequences and illuminate the relationship between sequence and dynamics. Two of our sequences, AT-all and GC-end, support out-of-register metastable states, and we commence our analysis with the role of these shifted states.

AT-all possesses the richest and most complex MSM of the four sequences by virtue of its repetitive nature, comprising a hybridized state (H), dissociated state (D), and four out-of-register shifted states (5S2, 3S2, 5S4, 3S4) (Fig.~\ref{fig:allseq_table}c). Analysis of the MSM transition probabilities reveal a critical role of the shifted states in mediating hybridization and dehybridization. Commencing from the dissociated state D, we observe approximately equal probabilities for transitions to each of the other five states, such that a transition to one of the out-of-register shifted states 5S2, 3S2, 5S4, or 3S4 is approximately 2.2 times more likely than a direct transition to the hybridized state H. Commencing from the hybridized state H, however, a direct transition to the dissociated state is approximately 1.2 times more likely than a transition to one of the two-base shifted states 5S2 or 3S2. Once in one of the four shifted states, the 5$^\prime$ vs. 3$^\prime$ overhang and degree of shifting play an important role in determining whether the duplex will transition to more shifted states, more aligned states, or completely dissociate. Transitions from more shifted states towards more aligned states (i.e., 5S4 $\rightarrow$ 5S2, 5S2 $\rightarrow$ H, 3S4 $\rightarrow$ 3S2, 3S2 $\rightarrow$ H) are approximately an order of magnitude more probable than the reverse transitions from more aligned states to more shifted states. The largest single transition probability from the four shifted states 5S2, 3S2, 5S4, and 3S4 is, however, back to the dissociated state D. Consistent with the higher destabilizing effect of 3$^\prime$ inert tails relative to 5$^\prime$ tails,~\citep{Doktycz1990ThermodynamicATGC, Dickman2012ThermodynamicDNAs, Michele2014EHybridization} the 3S4 $\rightarrow$ D transition probability is twice as large as the 5S4 $\rightarrow$ D, and the 3S2 $\rightarrow$ D is four times larger than the 5S2 $\rightarrow$ D. The transition probability from the 5S2 and 3S2 states back to the dissociated state D is equal to or greater than the transition probability to the hybridized state. A transition path theory analysis of the MSM reveals that 33\% of productive hybridization trajectories D $\dashrightarrow$ H (where the dashed arrow indicates the union of direct and indirect pathways) proceed through one or more out-of-register shifted states. Among these out-of-register pathways, the D $\dashrightarrow$ 5S2 $\dashrightarrow$ H transition is predicted to occur 57\% of the time. Mean first passage time (MFPT) analysis returns a MFPT for D $\dashrightarrow$ H of 3.0 $\mu$s and for H $\dashrightarrow$ D of 2.5 $\mu$s. The longer D $\dashrightarrow$ H time scale is expected given the number of out-of-register states that can form and re-dissociated before reaching H. These values are in close agreement with the leading implied time scale shown in  Fig.~\ref{fig:allseq_table}.  \noteb{can make note of the fact that Meta $\rightarrow$ (all other states) MFPTs align with the higher order modes as well? Not sure if this would require a more in depth discussion}

%Transition probabilities are higher when moving towards a more aligned state than towards a more shifted state -- e.g. 5S2 $\rightarrow$ H is more likely than 5S2 $\rightarrow$ 5S4 -- suggesting that more shifted states play a greater role in facilitating the hybridization process than dissociation. Furthermore, we see equal or higher transition probability from shifted states to the dissociated state (5S2 $\rightarrow$ D) than to more aligned states (5S2 $\rightarrow$ H), indicating that the shifting-hybridization process is frequently disrupted by complete dissociation. In particular, we observe that the transition probability 3S4 $\rightarrow$ D is 10x higher than the 3S4 $\rightarrow$ 3S2 path towards native hybridization. 

%Taken together, our results show that AT-all hybridization and dissociation kinetics are substantially modulated by out-of-register pathways. A majority of initial hybridization contacts occur out-of-register, and, based on transition probabilities, 35\% of D $\rightarrow$ H pathways and 37\% of H $\rightarrow$ D pathways pass through some out-of-register state. The 3S4 and 5S4 states are more prone to re-dissociation and thus contribute to about 5\% of these pathways. 

GC-end comprises the next most complex MSM with the introduction of the terminal G:C pairs maximally preserving the repetitive tract of A:T base pairings. The GC-end MSM possesses all of the same macrostates in its dynamical landscape with the exception of the 3S4 state (Fig.~\ref{fig:allseq_table}c). As discussed in Section \ref{sec:NN}, the 3S4 state is rendered unstable within the lag time of our MSM due to the presence of the destabilizing 3$^\prime$ inert tail and only four WC base pairings compared to six in the case of AT-all. Analysis of the transition probabilities reveal significant differences compared to those in the AT-all kinetic network. Commencing from the dissociated state D, we observe a similar transition probability to the 5S4 state as for AT-all, but once in the 5S4 state there are no significant transition probabilities to any other state except back to D. As such, the 5S4 state acts as a kinetic trap rather than as an intermediate to hybridization. The D $\rightarrow$ H and H $\rightarrow$ D transition probabilities are commensurate with those for AT-all. However, the D $\rightarrow$ 5S2 and D $\rightarrow$ 3S2 transition probabilities are half or less of those in AT-all, and the reverse transitions are an order of magnitude larger. This may be attributed to the reduced thermodynamic stability of the 5S2 and 3S2 states in GC-end that contain only six WC pairs compared to eight in AT-all (Fig.~\ref{fig:NN_table}). The 5S2 $\rightarrow$ H and 3S2 $\rightarrow$ H transition probabilities are more than an order of magnitude larger than in AT-all, which may again be attributed to the lower thermodynamic stability of the two shifted states relative to the hybridized state H. Again, the transition probabilities out of the 3S2 state to D or H are comparatively higher than those out of the 5S2 state, consistent with the increased destabilizing effect of 3$^\prime$ inert tails.~\citep{Doktycz1990ThermodynamicATGC, Dickman2012ThermodynamicDNAs, Michele2014EHybridization} Commencing from the hybridized state H, a direct transition to the dissociated state is approximately 6.8 times more likely than a transition to one of the two-base shifted states 5S2 or 3S2. A transition path theory analysis of the MSM reveals that only 7\% of hybridization events D $\dashrightarrow$ H and dehybridization events H $\dashrightarrow$ D proceed through one or more out-of-register shifted states. The significantly reduced role for out-of-register shifted states in mediating the hybridization and dissociation pathways for GC-end relative to AT-all is consistent with the reduced thermodynamic stability of these states due to the elimination of possible out-of-register WC base pairing for the terminal G:C pairs and therefore a reduced accessibility of these states in the GC-end kinetic network. A mean first passage time (MFPT) analysis returns a MFPT for D $\dashrightarrow$ H of 1.6 $\mu$s and for H $\dashrightarrow$ D of 2.1 $\mu$s, which are both notably lower than AT-all values. This may be attributed in part to shorter meta-stable lifetimes, or to the higher temperature at which this sequences was simulated. Again, the average of these MFPTs align well with the leading implied time scale values in Fig.~\ref{fig:allseq_table}.

%The 5S2 and 3S2 states still readily convert with H, however there is an order of magnitude lower probability of reaching these states from D when compared to AT-all. This suggests that inert tails inhibit out-of-register binding -- accounting in part for higher free energies discussed above -- but may not substantially disrupt out-of-register hybridization mechanisms along 5S2 $\rightarrow$ H and 3S2 $\rightarrow$ H once shifted states have formed.

%While GC-end strands can access out-of-register states, only 11\% of native hybridization events pass through any out-of-register states. These observations show that small disruptions to repetitive tracts shift hybridization pathways towards more two-state behavior. Indeed, there remain some repetitive tracts in GC-mix and GC-core, however we observe no out-of-register pathways for these sequences.

The out-of-register kinetic landscape that define AT-all and GC-end hybridization have been informed by numerous previous computational studies. Simulations have identified internal displacement mechanisms capable of correcting base pair alignment in 3SPN.2 \citep{Hinckley2014Coarse-grainedEffects} as well as in the coarse-grained oxDNA \citep{Romano2013DNADependence} and BioModi \citep{Markegard2015} models \noteb{not sure what you think of naming these specifically}. In all cases, these mechanisms were shown to be crucial components of the hybridization pathway for homogeneous and repetitive sequences. Xiao et al.\ performed an all-atom energy landscape-based analysis of 5$^\prime$-GGGGGG-3$^\prime$ and 5$^\prime$-GCGCGC-3$^\prime$ hexamers.~\citep{Xiao2019} Out-of-register states for 5$^\prime$-GCGCGC-3$^\prime$ hexamers were identified as deep kinetic traps along the hybridization pathway and ``slithering'' through these states did not provide a significant hybridization pathway compared to an alternative ``zippering'' mechanism. (In contrast, slithering through out-of-register shifted states and zippering served as two parallel pathways for hybridization of 5$^\prime$-GGGGGG-3$^\prime$.) This stands in contrast to our results for our AT-all (5$^\prime$-ATATATATAT-3$^\prime$) sequence, in which out-of-register states participated in 33\% of productive hybridization events. It is conceivable that the stronger hydrogen bonding in G:C WC pairs relative to A:T pairs may render out-of-register shifted states less favorable to hybridization by suppressing fluctuation-driven rearrangements,~\citep{Yakovchuk2006Base-stackingHelix,Zacharias2020Base-PairingFormation} but additional studies would be required to reconcile these observations.


\subsection{Central GC placement induces long-lived frayed states (GC-core)} \label{sec:GC-core}

The GC-core MSM represents a departure from the relatively rich and complex kinetic networks dominated by out-of-register shifted states to a much simpler one dominated by fraying (Fig.~\ref{fig:allseq_table}c). The MSM contains only three states -- hybridized H, dehybridized D, and frayed F4. The F4 state is unique to GC-core and contains up to six WC pairs -- the two central G:C core pairs and as many as four A:T pairs (it may be less due to rapid 1-2 terminal base pair fraying) on one side or other of the core; the other run of four A:T pairs remains free. (As expected by symmetry, the particular AT run that is made vs.\ free occurs with equal probability.) Our model reveals the absence of any direct hybridization or dehybridization transitions between the H and D states, with all pathways passing through the frayed state F4. Previous studies would suggest that hybridization of this sequence should proceed via a zippering mechanism, wherein upon formation of the strong central G:C WC base pairings the duplex helix should rapidly assemble in a middle-out fashion.~\citep{Romano2013DNADependence, Yin2011KineticsHybridization} Our results are partially consistent with this expectation, but reveal the frayed state F4 to be unexpectedly metastable, serving as a long-lived state with an mean life time of 1.8 ns. The stability of the state is attributable to the enthalpic stabilization offered by the six WC pairs and the entropic stabilization associated with the configurational entropy of the two free AT-tails.

Analysis of the transition probabilities show that commencing from the F4 state, progression to the hybridized state F4 $\rightarrow$ H is 25-times more likely than dissociation F4 $\rightarrow$ D. Thus once a D $\rightarrow$ F4 transition has occurred, a F4 $\rightarrow$ H transition will likely proceed; concomitantly, H $\rightarrow$ F4 events tend to fall back to the H state and are unlikely to proceed to complete dissociation. The transition probabilities H $\rightarrow$ F4 and D $\rightarrow$ F4 are, respectively, one half and one quarter as likely as the F4 $\rightarrow$ H transition probability \noteb{Not sure how meaningful this is considering they are coming from different source states?}. We have noted in Section \ref{sec:time} that fraying dynamics in the 3SPN.2 model appear to be significantly accelerated relative to center-of-mass translation, and it is conceivable that this may lead to elevated sampling of the F4 state within the computational model relative to experiment and the induction of more frequent dissociation. Moreover, since GC-core is the sequence most prone to fraying, this effect could be the root of the relatively poorer agreement of the $k_d^\mathrm{slow}$ response for GC-core compared to the other sequences due to an artificially elevated computational prediction of this rate (Fig.~\ref{fig:relaxation-comparison}a). Nevertheless, under these caveats our model predicts a MFPT for D $\rightarrow$ F4 $\rightarrow$ H of 3.4 $\mu$s and for H $\rightarrow$ F4 $\rightarrow$ D of 2.9 $\mu$s. The F4 state is unlikely to contribute significantly to the difference between MFPTs, rather we attribute this to a slight deviation in the total state populations -- the D equilibrium probability is about 9\% higher than H and F4 combined. Despite extensive sampling, these asymmetries arise from DNA's high temperature sensitivity and create the impression of a shorter H $\dashrightarrow$ D MFPT relative to D $\dashrightarrow$ H  \noteb{this should the origin of the discrepency (same for GC-mix) but I'm not sure if this is best way to explain it}.

Lattice models have previously identified frayed states as putative intermediates in DNA hybridization/dehybridization.\citep{Araque2016LatticeCooperativity, Phys2019,Sanstead2016} Araque et al.\ studied a 5$^\prime$-ATGCGCAT-3$^\prime$ octomer using a lattice model and identified a symmetrically A:T frayed state as a crucial part of the duplex transition path.\citep{Araque2016LatticeCooperativity} We previously studied the four sequences that are the subject of the present work using T-jump IR and 2D IR spectroscopy and identified GC-core as possessing the highest deviation from two-state behavior during dissociation when neglecting out-of-register contributions.\citep{Sanstead2016} This result was interpreted to arise from loss of A:T contacts and fraying around the central G:C core, and this hypothesis was supported by lattice model calculations that predicted the GC-core conformational ensemble to possess substantially more frayed configurations than the other three sequences.~\citep{Phys2019} Follow-up T-jump measurements and Smoluchowski simulations on model 1D free energy landscapes showed that AT termini fraying was an effectively barrierless process characterized by rapid inter-conversion between all accessible frayed states.\citep{Sanstead2018DirectDehybridization} These prior results are consistent with the present findings that expose the GC-core sequence to be the only sequence that occupies the F4 frayed state and therefore the only one possessing a metastable frayed state on time scales exceeding the 1.2 ns lag time of our MSMs.

%When examining all four sequences using T-jump IR and 2D IR spectroscopy, Sanstead et al. found that GC-core had the highest deviation from two-state behavior during dissociation (when excluding out-of-register contributions).\citep{Sanstead2016} As T-jump IR data only showed a loss of A:T contacts during the fast response, this intermediate state was defined by a high degree of fraying about the central core. While 1-2 base pair fraying was commonly observed for GC-mix and AT-all as well, their lattice model predicted GC-core to have substantially more frayed base pairs.\citep{Phys2019} Variable T-jump measurements and Smoluchowski simulations on model 1D free energy landscapes showed that AT termini fraying was an effectively barrierless process characterized by rapid inter-conversion between all accessible frayed states.\citep{Sanstead2018DirectDehybridization} We see the same rapid fraying in simulation data -- which is too fast to be attributed to a converged SRV mode -- however our SRV-MSM results show this inter-conversion first relies on the slower formation of the the A:T bond nearest to the GC center.  

%Although this process occurs much slower than single A:T base bonding and breaking, it may be difficult to experimentally discern from the overall hybridization process which contains both G:C and A:T character and occurs on a similar time scale.

%Our diffusion map analysis further shows how the ensemble of 4-bp frayed configurations impede helix formation.


\subsection{Disruption of repetitive AT tracts promotes two-state ``all-or-nothing'' kinetics (GC-mix)}

The GC-mix sequence is the only one of the four sequences studied that exhibits simple two-state ``all-or-nothing'' behavior.~\citep{Xiao2019, Araque2016LatticeCooperativity, Sikora2013ModelingIntermediates, Sanstead2016} The GC-mix MSM comprises just two states, the hybridized H and dehybridized D (Fig.~\ref{fig:allseq_table}c), indicating that association and disassociation of the strands proceeds directly without passing through any metastable intermediate states resolvable on the 1.2 ns time scale of our MSM. The two-state behavior appears to arise as a consequence of the placement of the G:C pair that maximally disrupts the repetitive AT tract within the decamer and destabilizing either out-of-register shifted states or frayed states. We note we do observe transient fraying of the terminal two-base AT tails within our dynamical simulations, but these frayed states are not sufficiently thermodynamically stable to produce a metastable macrostate within the resulting MSM. This stands in contrast to the metastable F4 state populated by GC-core, and can be understood by the reduced entropic stabilization offered by the configurational entropy of the frayed two-base AT tails relative to the frayed four-base AT-tails, and the more limited disruption of the helical duplex by the shorter tails that offers less kinetic inhibition to its reformation from the transiently frayed structures. Simulations of GC-mix were conducted at the melting temperature of the sequence, so the simple two-state MSM should have, by construction, equal populations of, and equal transition probabilities between, the H and D states. As we observed for GC-core, a higher dissociated state population produces a larger effective MFPT for D $\rightarrow$ H (2.9 $\mu$s) than H $\rightarrow$ D is (2.3 $\mu$s). Again, the average of these time scales match the dominant leading implied time scale for this sequence (Fig.~\ref{fig:allseq_table}c).

%Given the lack of a repetitive AT interior (as in AT-all and GC-end) or exterior (as in GC-core), we expect more canonical dynamics from GC-mix. Although GC-mix dynamics are most similar to those of GC-core, we did not observe a converged slow mode corresponding to multi-base fraying behavior for GC-mix. Instead, we observed two modes converge, corresponding to the association/dissociation dynamic and diffusive behavior while strands are dissociated, respectively. Given that this second mode did not inform the hybridization process, we designated this transitions as effectively two-state within the resolution of our model. We did, however, observe substantial fraying of the two AT termini in the simulation data. Although these frayed states may be too short-lived to resolve a distinct slow mode, this behavior shows qualitative agreement with experimental analysis of this sequence which attributed fraying prior to dissociation as a deviation from all-or-nothing behavior.\citep{Sanstead2016}  While AT-termini fraying is surely a prerequisite to dissociation, we find these states to be so common and transient that very few progress to full dissociation compared to the F4 state. Furthermore, one or two-base-pair fraying does not fundamentally disrupt the helix in such a way that its re-formation is kinetically inhibited by the intermediate structures we present for GC-core. 

Given the very simple two-state ``all-or-nothing'' behavior of GC-mix and the absence of any intermediate metastable states, we sought to interrogate our simulation trajectory data to elucidate the hybridization and dehybridization mechanisms. To do so, we followed all 10 intermolecular distances between native WC base pairs and tracked their evolution through a number of hybridization and dehybridization events. We present one representative example of each event in Fig.~\ref{fig:GC-mix_transitions} and four more of each in SI Figure 3. In hybridization, we observe a global decrease in all 10 distances as the strands approach one another, but uniformly observe the formation of some key native WC base pair contacts immediately prior to duplex formation. Specifically, formation of one of the G:C WC pairs and at least one neighboring A:T pair or 2-3 central A:T pairs \noteb{this is a more general observation which is in line with all events} This behavior is consistent with a ``nucleation-zippering'' mechanism as has been reported in previous studies.\citep{Wetmur1968KineticsDNA, Porschke1971CooperativeTransition, Sambriski2009SequencePathways,  Yin2011KineticsHybridization} In dehybridization, we observe fraying on the two-base AT-tails on one or both sides of the duplex followed by rapid dissociation of the central WC base pairs. Qualitatively, we observed some short-lived states composed of two to four native WC base pair contacts immediately before full dissociation occurs, but, in contrast to the F4 state we observe in GC-core, these conformations do not constitute a metastable state within our MSM nor do they tend to reform intact duplexes. These dissociation dynamics are similar to previously reported ``fraying-peeling'' mechanism.\citep{Wong2008TheSimulations, Perez2010Real-timeUnfolding, Zgarbova2014BaseRNA} 

\begin{figure}[ht!]
	\begin{center}
	\includegraphics[width=180mm, scale=1]{Fig4.pdf}
        \caption{GC-mix hybridizes by nucleation-zippering and dehybridizes by fraying-peeling. Tracking of the 10 intermolecular distances between native WC base pairs over the course of an (a) hybridization event and (b) dehybridization event. Symmetrically permutable distances (e.g., A$_1$:T$_{10}$ and T$_{10}$:A$_1$) are reflected across the x-axis to avoid congestion in the plot. Circles superposed on the x-axis indicate the instantaneous MSM state assignment as dissociated D (blue) or hybridized H (orange). Hybridization tends to occur by a nucleation-zippering mechanism, wherein a native G:C WC pair and adjacent A:T pair or central A:T pairs first form prior to rapid formation of the complete duplex. Dehybridization tends to occur by a fraying-peeling mechanism wherein fraying of the two-base AT-tails on one or both sides of the duplex precedes dissociation of the central native base pairs and complete dissolution of the duplex. Four additional hybridization events and four additional dehybridization events are presented in SI Figure 5.
        }
        \label{fig:GC-mix_transitions}
	\end{center}
\end{figure}






\subsection{\label{sec:stretch_comparison}T-jump IR responses are correlated with the complexity of the dynamical landscape}

We hypothesized that the complexity of the MSM kinetic landscapes would be reflected in T-jump IR measurements and sought experimental validations for the predictions of our molecularly-detailed kinetic models. As discussed in Section \ref{sec:time}, the fast and slow IR responses correspond well to terminal base fraying and dissociation, respectively. Although there are no distinct ``middle'' responses at the expected shifting timescales, the response profile can inform the distribution of dynamics contributing to the signal \citep{Hamm2006StretchedFolding, Johnston2006StretchedDecays}. In particular, the AT-all fast response exhibits more stretched exponential character ($\beta$ = 0.3) than any of the other three sequences (Figure \ref{fig:T-jump_stretching}). We can interpret the stretch as fraying from many configurations, each having a slightly different rate. The greater fraying heterogeneity is in agreement with the diverse kinetic landscape of the AT-all MSM. For the GC-end response, however, the stretching factor is not appreciably different from the other sequences and comparison of 1660 and 1600 cm$^{-1}$ wavelengths indicate that the fast response is much stronger in the G/T rather than A/T region. This further suggests that it corresponds to terminal fraying rather than shifting. Overall, there are several reasons why the T-jump response may not be directly sensitive to shifting. In the case of GC-end, the low population of shifted states alone may diminish contributions below a detectable limit. For AT-all, the particularly slow timescales associated with the most common shifting process (H $\leftrightarrow$ 5S2) can blend in with the slow dissociation response. In both cases, it is unclear how populations of shifted states change with temperature, making these processes difficult to identify with T-jump techniques. 

These comparisons also highlight the importance of considering and defining multiple types of deviations from two-state behavior. In this work, we define this deviation by the complexity of the MSM network and number of accessible metastable states. This definition suggests the following order of kinetic complexity: AT-all $>$ GC-end $>$ GC-core $>$ GC-mix. As we have shown, this trend is closely related to sequence repetitiveness, specifically the number of consecutive AT motifs. In our previous work, we did not consider out-of-register states as part of the experimental or lattice model analysis, and therefore determined our ordering (GC-core $>$ GC-mix $>$ AT-all $>$ GC-end) in terms of frayed state populations prior to duplex dissociation\citep{Sanstead2016, Sanstead2018DirectDehybridization}. Depending on the biological system or nanotechnology application of interest, these forms of deviation may hold varied importance \noteb{could add example here, but don't want to dwell on this too much?}.

%In addition to identifying relaxation time scales, the T-jump IR response profile can inform the distribution of dynamics contributing to the response. In particular, the fast response of AT-all and GC-end exhibits more stretched exponential character than for GC-core and GC-mix  (Figure \ref{fig:Fig6}). Furthermore, the GC-end response contains both G:C and A:T vibrational contributions, suggesting that native terminal base fraying may not be the unique source of the response. On the other hand, GC-mix and GC-core are characterized by a distinct fast response comprised only of A:T features that points to A:T fraying as the predominant dynamic source. We predict that the mid-IR responses of fraying and shifting are not clearly distinguishable for GC-end and AT-all, therefore we cannot determine the stretched responses originate from fraying dynamics in out-of-register configurations, shifting mechanisms during the dissociation process, or some combination of the two. Although reproducing these stretched signals is not within the scope of this analysis, we see qualitative agreement between more complex transition network (AT-all $>$ GC-end $>$ GC-core, GC-mix) and increased stretching in the fast response. 

% We expect that fraying would still occur in out-of-register conformations but at a different rate than intact fraying.

%It is difficult to distinguish a collection of different starting configurations -- such as out-of-register duplexes -- and their associated dynamics prior to the temperature jump from mechanistic pathways after the jump.

%We emphasize that this does necessarily reflect shifting relaxation time scales themselves but more likely changes in the fast responses -- e.g. fraying out-of-register -- within the collection of metastable states. 

\begin{figure}[ht!] % right place for NN table? could go to SI
	\begin{center} 
        \includegraphics[width=175mm, scale=1]{Fig5.pdf}
        \caption{T-Jump IR responses reflect sequence-dependent conformational heterogeneity. (a) Mid-IR t-HDVE difference spectra for GC-core at time delays from 5 ns to 560 $\mu$s. Normalized time traces for each sequence are shown at (b) 1600 cm$^{-1}$ and (c) 1660 cm$^{-1}$. The signal at 1600 cm$^{-1}$ corresponds to changes in A and T ring vibrations while the signal at 1660 cm$^{-1}$ contains contributions from G and T carbonyl vibrations. Each time trace is fit to the sum of stretched exponential with two exponentials (solid lines): $S(t) = A\exp{(-(t/\tau^{\mathrm{fast}})^{\beta})} + B\exp{(-t/\tau_{\mathrm{slow}})}+C\exp{(-t/\tau_{\mathrm{cool}})}$. The stretched exponential describes the process from 5 ns to 1 $\mu$s, and the two exponentials describe the signal increase from 1-320 $\mu$s and signal decay from re-hybridization induced by thermal relaxation back to the initial temperature. The stretch factor $\beta$ from each fit is listed.}

        \label{fig:T-jump_stretching}
	\end{center}
\end{figure}

   
   
   

%\subsection{5S2 state distributions reveal differences between AT-all and GC-end kinetics}

%Although our SRV-MSMs provide useful insight into slow processes, we were also interested in the ensemble of configurations within PCCA+ macrostates and during transition periods between states. These configurations interchange significantly faster than the SRV-MSM lag time, but we can use a diffusion map approach to glean a structural details of the configuration in these regions.\citep{Coifman2006DiffusionMaps, Ferguson2010SystematicMaps} While examining molecular renderings, we noticed that a significant proportion of GC-end 5S2 configurations retained one native G:C bond, even when all available A:T bonds were formed out-of-register. We would not necessarily expect duplexes to sacrifice helical conformational entropy in order to facilitate termini bonding. To compare how these state populations differ between GC-end and AT-all, we employed diffusion maps built on an equal sampling of 5000 conformations from the 5S2 state of both sequences. We used all 100 intermolecular distances (as opposed to the 55 permutation free coordinates used to construct SRV-MSM) as our distance metric, making it easier to discern structures that form on either permutable end of the shifted conformation. This created degenerate 2nd and 3rd diffusion modes, with nearly equal eigenvalues, differentiating effects at the identical ``top'' and ``bottom'' of the strands. In Figure \ref{fig:GC-end_dmaps} we present the first two non-trivial diffusion map eigenfunctions and show representations of the degenerate third coordinate in the SI. Diffusion maps built from samples of the 3S2 states are also shown in the SI. 

%\begin{figure}[ht!]
%	\begin{center}
%        \includegraphics[width=150mm, scale=1]{Fig8.pdf}
%        \caption{First two diffusion map coordinates built from 10000 5S2 states, equally sampled from AT-all and GC-end. Color maps show inverse distances (nm$^{-1}$) between out-of-register ends and complementary ends. The lower right region of the diffusion map space shows a high concentration of ``shifted loop'' base pairs that may facilitate the out-of-register GC-end transition.}
%        \label{fig:GC-end_dmaps}
%	\end{center}
%\end{figure}

%The first diffusion mode clearly delineates between the GC-end and AT-all shifted conformations and correlates highly with the average distance between the 3$^\prime$ end and its shifted complementary pair. As expected, this shows that mismatched C:T pairs are never bound whereas the AT-all pairs are mostly bound with occasional fraying indicated by small AT-all overlap in the GC-end region. The second diffusion mode, which correlates highly with the average distance between 3$^\prime$ and 5$^\prime$ ends, highlights structural variances within the two sequences distributions. We notice that about 40\% of GC-end configurations fall below $\Psi_3 = -0.01$  compared to only 10\% of AT-all configuration. We find that these GC-end populations share stabilizing termini contacts despite all A:T contacts being shifting out-of-register. These ``shifted-loop'' bonds appear uniquely stable for GC-end conformations, and suggest a transition state through which hybridization is facilitated. This may account for the GC-end 5S2 $\rightarrow$ H transition being more favorable than  5S2 $\rightarrow$ D despite the state free energy being substantially higher than the equivalent AT-all pathways.

%This effect may be exaggerated given that C and T base pairs are assigned slightly higher excluded volume radii in 3SPN.2 \citep{Hinckley2013AnHybridization}. 

%Although AT-all shifted ends tend to stay bound out-of-register, the second diffusion coordinate shows some population of inert tails that fold back onto the helix. 

% need to work on this if adding to this section (might belong somewhere else or not at all?
%Although internal base pair mismatches can cause substantial conformational distortions such as kinking, terminal mismatches have been shown to be slightly stabilizing and have a minimal effect on helical character \citep{Santalucia2004TM, DiMichele2014EffectHybridization}. In the context of the 3SPN.2 model, these ends are accounted for via intra-strand base stacking and inter-strand cross-stacking interactions \citep{Hinckley2013AnHybridization}. The only direct interaction between non Watson-Crick (WC) basepairs is parameterized by isotropic excluded volume potential, which is likely more simplistic than the true mismatch interaction. 

%AT-all looping could be a consequence of multiple bps sharing the same wc partner (I believe this is possible in 3SPN.2, because types of bonds are only mutually exclusive between cross-stacking, bp binding, and excluded volume)%







%\subsection{Sequence-dependent fraying determines direct hybridization pathways} 

%\begin{figure}[ht!]
%	\begin{center}
%        \includegraphics[width=150mm, scale=1]{Fig9.pdf}
%        \caption{Sequence-dependent population distributions for trimmed hybridization events. $\Psi_2$ represents the maximum structural variance across all sequence configurations and correlates strongly with hybridization progression. Populations are discretized in 200 bins and smoothed by averaging with the nearest 5 bins. Heat maps show the average intermolecular base distances in nm at increasing values of $\Psi_2$, where the main diagonal shows native base pair distances.}
%        \label{fig:Fig8}
%	\end{center}
%\end{figure}

%To compare fast hybridization dynamics across all sequences, we trimmed simulations centered on direct D$\rightarrow$H (or D$\rightarrow$F4$\rightarrow$H for GC-core) events and excluded transitions that passed through long-lived out-of-register states. To achieve higher resolutions, we reran selected simulation with a 10x higher save rate and analyzed a 1000 frame region (10 ns simulation time) for each event. We built diffusion maps on a set of 48 such ``trimmed'' events evenly sampled across sequences. We found a distinct spectral gap after the first non-trivial diffusion map eigenvalue, indicating that the leading mode $\Psi_2$ contained most of the structural variance. Indeed, $\Psi_2$ had a 0.89 Pearson correlation with simulation time and a  -0.94 correlation with inter-strand core distance with minimal variation by sequence. This shows that $\Psi_2$ is strongly correlated with hybridization progress for all sequences and represents a consistent collective variable for comparison between sequences. 

%Once we determined a promising hybridization coordinate, we compared probability distributions for each sequence along $\Psi_2$ (Figure \ref{fig:Fig8}). The region where $\Psi_2 < 0$ is noisy and mainly reflects the distance between approaching strands, but we noted statistically meaningful differences between sequence populations once first Watson-Crick contacts are made. In the $ 0.001 <\Psi_2 < 0.0035$ region, heatmaps show that contacts tend to form near the center of the duplex, and frayed structures progressively zipper closed as $\Psi_2$ increases. As expected, GC-core shows a distinct peak in the frayed region, corresponding to the F4 macrostate identified by SRV-MSMs. GC-mix has a notably higher population than AT-all and GC-end but displays a more gradual progression toward the native hybridized state. This is consistent with the downhill nucleation-zippering mechanism described above. Although differences between AT-all and GC-end are more subtle, overall deviations from two-state behavior follow predictions by Sanstead et. al.\citep{Sanstead2016}  where GC-core > GC-mix > AT-all > GC-end. This qualitative agreement shows that MSM information can be adapted to compare sequences along common pathways and elucidate dynamics occurring faster than the lag time. Furthermore, we are able to distinguish out-of-register deviations from fraying deviations and describe relevant experimental comparisons for each.

%This time, we set our distance metric to the same permutation-free coordinates we used to build the SRV-MSMs. 

% add new info here, not sure if shifting discussion is necessary
% take away message is that these are good proxies for the responses
% although there is not perfect agreement, we show that these are very reasonable coordinates assing fast and slow response









%\subsection{Limitations}

%Although we were able to obtain improved resolutions on several relevant dynamics by using a shared lag time across sequences, we found it difficult to converge faster processes such as duplex nucleation and zippering. These processes are crucial to duplex formation, however they do not appear to be kinetically metastable or slow relative to other time scales of interest. Furthermore, they can initiate at various points along the strand, which, under out present featurization method, may appear as a collection of modes instead of as one distinct process. We also found the need to strike a balance between adequate sampling of hybridization events and frame save rate in order to maintain tractable SRV-MSM calculations. Indeed, we collected over 10 GB of equilibrium trajectory data for each sequence and were working near memory limits when training SRVs and building SRV-MSMs.

%In any high-level model there are inevitable simulation artefacts produced by coarse-grained approximations. For example, the treatment of non-interacting base pairs as an excluded volume potential alone may not be representative of dynamics produced from mismatched dangling ends. In general, coarse-grained models produce a smoother free-energy surface which can results in much faster motions between states. This is illustrated by substantial accelerations in temperature-dependent responses when compared to experiment. Furthermore, it may be easier to cross between states -- e.g. a 5S2 $\rightarrow$ H transition -- when the usually rough free energy path becomes more easily traversed. 




\section{\label{sec:conc}Conclusions}

We have conducted an integrated computational and experimental study of the sequence-dependent kinetic mechanisms for the hybridization and dehybridization dynamics of a family of four self-complementary 10-mer DNA oligomers: ATATATATAT (AT-all), GATATATATC (GC-end), ATATGCATAT (GC-core), and ATGATATCAT (GC-mix) \noteb{again, do we want keep the 5$^\prime$-ATATATATAT-3$^\prime$ format here or not bother}. We conducted more than 1 ms of unbiased coarse-grained molecular dynamics simulations and employed deep learning techniques to construct high-resolution Markov state models as predictive and interpretable models of the sequence dependent dynamics. T-jump IR spectroscopy was used to calibrate the kinetic time scales of the coarse-grained molecular model and validate the predictions of the Markov state models. Our results reveal that the specific placement of interrupting G:C pairs within an otherwise repetitive AT sequence can have a profound impact on the kinetic pathways and mechanisms for association and dissociation of the DNA duplex. In particular, we found AT-all to possess the richest and most complex kinetic landscape of the four sequences that is dominated by out-of-register shifted states that participate in 33\% of hybridization and dehybridization events. These results are in accord with observations from HDVE T-jump IR spectroscopy that a spectrum of fast fraying responses likely originating from an ensemble of out-of-register states. Introduction of the G:C pairs at the end of the strand maintains a eight-base pair repetitive AT tract and the GC-end kinetic landscape possess all but one of the same out-of-register shifted states as AT-all. Similar to AT-all, \rood{XX}\% and \rood{XX}\% \noteb{not sure if we want to change the structure here given that these values appear to be the same (and are each only 7\%)} of hybridization and dehybridization events, respectively, in GC-end proceed through one or more shifted states. Placing the G:C pairs in the center of the strand maintains two four-base AT tracts either side of the core and results in qualitatively different kinetic behaviors for GC-core. In this case, no metastable out-of-register shifted states are observed in our models with the hybridization and dehybridization pathways all passing through a strongly metastable frayed state in which one or other of the four-base AT-tracts is unbound resulting in two free AT-tails. Finally, placing the G:C bases between the center and end of the strand to maximally disrupt the repetitive AT tracts results in no metastable out-of-register or frayed states and results in simple two-state ``all-or-nothing'' hybridization/dehybridization behavior. Analysis of the specific pathways reveals hybridization to largely proceed by a nucleation-zippering mechanism and dehybridization to proceed by a fraying-peeling mechanism. The ordering of the computationally predicted kinetic landscapes from most to least complex -- AT-all $>$ GC-end $>$ GC-core $>$ GC-mix \noteb{not sure exactly what to say here given that Section \ref{sec:stretch_comparison} is different now}

In sum, our results demonstrate the profound effect of sequence upon the kinetic landscapes, metastable states, and hybridization/dehybridization mechanisms of short DNA oligomers. Our analysis of this small family of sequences expose preliminary design principles for the (meta)stability of out-of-register and frayed states but we anticipate much greater richness in the landscapes will emerge with studies of longer and more diverse sequences. Going forward, we will extend this work to discern more general trends in sequence-dependent hybridization/dehybridization for a wider range of oligomer sequences and motivate strategies for experimental comparisons. We anticipate that these insights may provide foundational design rules by which to improve understanding of \emph{in vivo} hybridization processes and rationally engineer optimized sequences for applications such as DNA-PAINT,~\citep{Strauss2020UpDNA-PAINT} DNA barcoding,~\citep{Shah2019} and novel developments in dynamic DNA nanotechnology. 

%We have demonstrated how G:C placement in an otherwise repetitive AT sequence has a profound impact on equilibrium hybridization dynamics. By supplementing coarse-grained MD with data-driven time-lagged analysis, we constructed high resolution SRV-MSMs capable of distinguishing sequence-dependent kinetic behavior. In particular, we found that AT-all and GC-end sequences both participate in some degree of out-of-register base pair shifting, although these dynamics are more relevant for AT-all. On the other hand, GC-core hybridization transits through, or is perhaps facilitated by, a frayed intermediate in which one half of A:T bonds are broken and the duplex is significantly disrupted. Our computational approach and results show strong comparisons to T-jump experiments at ns and us time scales. Going forward, we expect that this work will be extended to predict more general trends in sequence-dependent hybridization and motivate strategies for experimental comparisons. These insights should be leveraged for enhanced understanding of \emph{in vivo} hybridization processes, optimized sequence design in application such as DNA-PAINT and barcoding, and novel developments in the growing field of dynamic DNA nanotechnology. 


\clearpage
\newpage

%\bibliographystyle{unsrt}
\bibliography{references_210505}

%\bibliography{refs_mendeley2}

%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% The "tocentry" environment can be used to create an entry for the
%% graphical table of contents.
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%

\clearpage
\newpage

\begin{figure}[ht!]
	\begin{center} 
        \includegraphics[width=150mm, scale=1]{TOC.pdf}
        \caption*{TOC image \noteb{could consider adding an experimental aspect to this?}}
        \label{fig:TOC}
	\end{center}
\end{figure}

\end{document}